<!DOCTYPE html>
<html>
    
    <!--
	Ion by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<head>

    <title>Phylogenetic analysis using RAxML</title>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <meta name="description" content="" />
    <meta name="keywords" content="" />
    <!-- Favicons -->
    <link rel="shortcut icon" type="image/png" href="http://localhost:4000/favicon.png" />
    <link rel="shortcut icon" href="http://localhost:4000/favicon.ico" />
    <!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
    <script src="/js/jquery.min.js"></script>
    <script src="/js/skel.min.js"></script>
    <script src="/js/skel-layers.min.js"></script>
    <script src="/js/init.js"></script>
    <noscript>
        <link rel="stylesheet" href="/css/skel.css" />
        <link rel="stylesheet" href="/css/style.css" />
        <link rel="stylesheet" href="/css/style-xlarge.css" />
    </noscript>
</head>


	<body id="top">

        <!-- Header -->
<header id="header" class="skel-layers-fixed">
    <h1><a href="/" class="site-title" >Moscou Group at The Sainsbury Laboratory</a></h1>
    <nav id="nav">
        <ul>
            <li><a href="/">Home</a></li>
	    <li><a href="/Research">Research</a></li>
	    <li><a href="/Publications">Publications</a></li>
	    <li><a href="/Funding">Funding</a></li>
	    <li><a href="/Blog">Blog</a></li>
	    <li><a href="/comics">Comics</a></li>
	    <li><a href="/CV">CV</a></li>
	    <li><a href="/Group/index.html">Group</a></li>
        </ul>
    </nav>
</header>


        <div class="wrapper style1">
    <article class="container">
        <header class="major">
            <h2 class="post-title">Phylogenetic analysis using RAxML</h2>
                <p class="post-meta">May 21, 2017</p>
        </header>

        <section class="post-content">
            <p>This tutorial describes the general approach of phylogenetic tree construction using RAxML, focusing on a example using the thioredoxin gene family in <em>Arabidopsis thaliana</em>. The analysis will incorporate the use of domain identification, domain extraction, multiple sequence alignment, phylogenetic tree construction, bootstrap analysis, and biological interpretation.</p>

<p>Examples files used in this analysis can be found on Github in the examples folder of <a href="https://github.com/matthewmoscou/QKdomain">QKdomain</a>.</p>

<h2 id="identification-of-thioredoxins-in-arabidopsis-thaliana">Identification of thioredoxins in <em>Arabidopsis thaliana</em></h2>
<p>The thioredoxins of <em>Arabidopsis thaliana</em> were identified by identifying all proteins with the Pfam domain PF00085. Next, InterProScan was used to evaluate these sequences. The resulting output in tab-separated format (tsv) is used by the <a href="https://github.com/matthewmoscou/QKdomain">QKdomain</a> (v1.0) suite of scripts to identify the diversity of domains. Initially, <code class="highlighter-rouge">QKdomain_preprocess.py</code> is used to assess the domain content present in the protein data set.</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>python QKdomain_preprocess.py Athaliana_167_TAIR10.protein.TRX.fa.tsv At_TRX_preprocess.txt
</code></pre>
</div>

<p>The output file <code class="highlighter-rouge">At_TRX_preprocess.txt</code> is used to manually curate identifiers to generate redundant abbreviations for the next component of the analysis. Below is an example of a set of abbreviations generated from <code class="highlighter-rouge">At_TRX_preprocess.txt</code>.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>Coil	CC
PF00085	TRX
PF00462	GLRX
PF00515	TPR
PF00789	UBX
...
</code></pre>
</div>

<p>Next, we identify the domain structure of proteins in the data set.</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>python QKdomain_process.py Athaliana_167_TAIR10.protein.TRX.fa Athaliana_167_TAIR10.protein.TRX.fa.tsv At_TRX_abbreviations.txt At_TRX_process.txt
</code></pre>
</div>

<p>The output will be structured with the first column the protein identifier and the second column the protein domain structure.</p>

<div class="highlighter-rouge"><pre class="highlight"><code>AT1G35620.1	TRX-TRX
AT3G56420.1	TRX
AT5G66410.1	TRX
AT1G14570.1	UBA-TRX-UIM-UBX-UBI
AT3G53220.1	TRX
AT3G15360.1	TRX
AT2G32920.1	TRX-TRX-TRX
AT1G08570.1	CC-TRX
...
</code></pre>
</div>

<p>To export all thioredoxin (TRX) domains in the set of proteins, use the following command:</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>python QKdomain_process.py -d TRX Athaliana_167_TAIR10.protein.TRX.fa Athaliana_167_TAIR10.protein.TRX.fa.tsv At_TRX_abbreviations.txt At_TRX_process.txt At.TRX.domain.fa
</code></pre>
</div>

<p>The file <code class="highlighter-rouge">At.TRX.domain.fa</code> will contain individual non-overlapping domains delineated by InterProScan. Overlapping domains would be integrated into a single domain and would require manual curation in a multiple sequence alignment to correct. The protein identifier is modified to include the start and end positions selected and the domain structure of the original protein.</p>

<h2 id="multiple-sequence-alignment-of-thioredoxin-domains-from-arabidopsis-thaliana">Multiple sequence alignment of thioredoxin domains from <em>Arabidopsis thaliana</em></h2>
<p>A multiple sequence alignment is critical for the development of an accurate phylogenetic tree. Several forms of software are available, and each has different strengths according the the size and complexity of the data set under investigation. A summary table of commonly used software is listed below (<strong>Table 1</strong>). A considerable number of additional options are available and this list should just be considering a starting point. In addition, parallel versions exist for several software, such as ClustalOmega and MUSCLE.</p>

<p><strong>Table 1.</strong> Multiple sequence alignment software</p>

<table>
  <thead>
    <tr>
      <th style="text-align: left">Software</th>
      <th style="text-align: center">Scalability</th>
      <th style="text-align: left">Complexity</th>
      <th style="text-align: left">Website</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: left">ClustalW</td>
      <td style="text-align: center">Small to Medium</td>
      <td style="text-align: left">Diverse sequences (more conservative)</td>
      <td style="text-align: left"><a href="http://www.clustal.org/clustal2/">http://www.clustal.org/clustal2/</a></td>
    </tr>
    <tr>
      <td style="text-align: left">ClustalOmega</td>
      <td style="text-align: center">Large</td>
      <td style="text-align: left">Diverse sequences (more relaxed)</td>
      <td style="text-align: left"><a href="http://www.clustal.org/omega/">http://www.clustal.org/omega/</a></td>
    </tr>
    <tr>
      <td style="text-align: left">MAFFT</td>
      <td style="text-align: center">Medium to Large</td>
      <td style="text-align: left">Â </td>
      <td style="text-align: left"><a href="http://mafft.cbrc.jp/alignment/software/">http://mafft.cbrc.jp/alignment/software/</a></td>
    </tr>
    <tr>
      <td style="text-align: left">MUSCLE</td>
      <td style="text-align: center">Medium</td>
      <td style="text-align: left">Diverse sequences (more conservative)</td>
      <td style="text-align: left"><a href="http://www.drive5.com/muscle/">http://www.drive5.com/muscle/</a></td>
    </tr>
    <tr>
      <td style="text-align: left">PRANK</td>
      <td style="text-align: center">Medium</td>
      <td style="text-align: left">Low complexity (highly related sequences)</td>
      <td style="text-align: left"><a href="http://wasabiapp.org/software/prank/">http://wasabiapp.org/software/prank/</a></td>
    </tr>
  </tbody>
</table>

<p>The thioredoxins of <em>Arabidopsis thaliana</em> are fairly diverse, therefore MUSCLE was selected for multiple sequence alignment. The first step is to identify a conserved region in the thioredoxin domain and removal of sequences unrelated to the thioredoxin domain and/or without conservation in this gene family.</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>muscle -in At.TRX.domain.fa -clwstrict -out At.TRX.domain.MUSCLE1.aln
</code></pre>
</div>

<p>A clear cutoff can be identified between positions 382 to 581 aa in the alignment. Next, we realign the extracted sequence, which can improve the assembly once divergent sequences are removed.</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>muscle -in At.TRX.domain.MUSCLE1_382_581.fa -clwstrict -out At.TRX.domain.MUSCLE2.aln
</code></pre>
</div>

<p>While the first alignment and selection will remove the majority of outlier sequence, it is important to check and remove any additional sequence. In this case, an extraction was made between the positions 34 and 217 aa in the multiple sequence alignment.</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>muscle -in At.TRX.domain.MUSCLE2_34_217.fa -clwstrict -out At.TRX.domain.MUSCLE3.aln
</code></pre>
</div>

<p>The last step involves an iterative process of removing sequences that are highly divergent from the other sequences in the alignment. This is performed by (1) performing the multiple sequence alignment, (2) construction of a neighbor joining tree, and (3) identification of genes that are highly divergent based on the tree structure. The details of the removed sequences are detailed below (<strong>Table 2</strong>).</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>muscle -in At.TRX.domain.MUSCLE2_34_217_curated.fa -clwstrict -out At.TRX.domain.MUSCLEf.aln
</code></pre>
</div>

<p><strong>Table 2.</strong> Manual curation of sequences in multiple sequence alignment based on aberrant phylogenetic inference.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center">Iteration</th>
      <th style="text-align: center">Gene</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center">1</td>
      <td style="text-align: center">AT1G20225.1_8_210</td>
    </tr>
    <tr>
      <td style="text-align: center">1</td>
      <td style="text-align: center">AT1G35620.1_180_327</td>
    </tr>
    <tr>
      <td style="text-align: center">2</td>
      <td style="text-align: center">AT4G04950.1_153_256</td>
    </tr>
    <tr>
      <td style="text-align: center">3</td>
      <td style="text-align: center">AT1G60420.1_188_299</td>
    </tr>
    <tr>
      <td style="text-align: center">3</td>
      <td style="text-align: center">AT3G54960.2_208_518</td>
    </tr>
    <tr>
      <td style="text-align: center">4</td>
      <td style="text-align: center">AT4G14250.1_500_624</td>
    </tr>
    <tr>
      <td style="text-align: center">4</td>
      <td style="text-align: center">AT4G14250.2_105_191</td>
    </tr>
  </tbody>
</table>

<p>An example of how the tree is incorrect is seen below.</p>

<p><img src="/assets/2017-05-21-phylogenetic_analysis/At.TRX.domain.MUSCLE2.phylogeny.png" alt="alt text" title="TRX phylogenetic tree without curation" /></p>

<p>After removal of divergent sequence, the tree takes its correct structure.</p>

<p><img src="/assets/2017-05-21-phylogenetic_analysis/At.TRX.domain.MUSCLE3.phylogeny.png" alt="alt text" title="TRX phylogenetic tree with curation" /></p>

<h2 id="phylogenetic-tree-construction-of-thioredoxin-domains-from-arabidopsis-thaliana">Phylogenetic tree construction of thioredoxin domains from <em>Arabidopsis thaliana</em></h2>
<p>Several approaches and software are available for phylogenetic tree construction including but not limited to parsimony, neighbor-joining, and maximum likelihood. Here, we demonstrate how to generate a maximum likelihood phylogenetic tree using <a href="https://github.com/stamatak/standard-RAxML">RAxML</a>. First, we generate the phylogenetic tree. In the first instance, we will use the automatic selection of amino acid selection model (PROTGAMMAAUTO) and use four processors <code class="highlighter-rouge">-T 4</code>.</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>raxml -s At.TRX.domain.MUSCLEf.phy -n At.TRX.domain.PROTGAMMAAUTO -m PROTGAMMAAUTO -p 84381764921 -T 4
</code></pre>
</div>

<p>The output will initially describe the set of sequences that are identical such as:</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>IMPORTANT WARNING: Sequences AT1G14570.1_163_295 and AT1G14570.3_142_274 are exactly identical


IMPORTANT WARNING: Sequences AT2G15570.2_46_174 and AT2G15570.1_46_173 are exactly identical


IMPORTANT WARNING: Sequences AT1G07700.3_65_214 and AT1G07700.1_52_201 are exactly identical
</code></pre>
</div>

<p>It will then perform automatic protein model assignment until it identifies an appropriate model. In this case, WAG. The final output will be in the file <code class="highlighter-rouge">RAxML_result.At.TRX.domain.PROTGAMMAAUTO</code>. The run time was 443 seconds using four processors on a relatively new MacBook Pro. This is important, as the length of the alignment and number of sequences will have a tremendous impact on the time required for phylogenetic tree construction. We routinely use large multicore servers to handle large data sets.</p>

<p>Next, we need to generate bootstraps to infer relationships of genes within the tree. This will take a considerable amount of time, but we can test for convergence as we perform blocks of bootstrap analyses.</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>raxml -s At.TRX.domain.MUSCLEf.phy -n At.TRX.domain.PROTGAMMAAUTO_bootstrap_r1 -N 100 -m PROTGAMMAWAG -p 427482396541 -T 4
raxml -s At.TRX.domain.MUSCLEf.phy -n At.TRX.domain.PROTGAMMAAUTO_bootstrap_r2 -N 100 -m PROTGAMMAWAG -p 9567154133 -T 4
cat RAxML_result.At.TRX.domain.PROTGAMMAAUTO_bootstrap_r<span class="k">*</span> &gt; allBootstraps
raxml -z allBootstraps -m PROTGAMMAWAG -I autoMRE -n TEST -p 3824142315 -T 4
</code></pre>
</div>

<p>On my machine, it tooks 5.03 hours to run the first 100 bootstraps. For testing this tutorial on your own computer, I would suggest preparing a small data set in the first instance. After 10 hours of running, the convergence test was met after 200 bootstraps. To incorporate bootstraps into the maxmimum likelihood tree, we used the following command:</p>

<div class="language-bash highlighter-rouge"><pre class="highlight"><code>raxml -f b -z allBootstraps -t RAxML_result.At.TRX.domain.PROTGAMMAAUTO -m PROTGAMMAWAG -n At.TRX.domain.PROTGAMMAWAGandBOOTSTRAP.txt
</code></pre>
</div>

<p>The file <code class="highlighter-rouge">At.TRX.domain.PROTGAMMAWAGandBOOTSTRAP.txt</code> can be directly used with <a href="http://itol.embl.de">EMBL iTOL</a> for visualization.</p>

<h2 id="files-used-in-all-analyses">Files used in all analyses</h2>
<p>All files used in this analysis can be accessed on my webpage source account on <a href="https://github.com/matthewmoscou/matthewmoscou.github.io/tree/master/assets/2017-05-21-phylogenetic_analysis">Github</a>.</p>

        </section>
        
        <footer>
            
            <a href="/comics/2017/05/17/shiny.html" class="float-left">&larr; Previous post</a> 
            
        </footer>

    </article>
</div>


	<!-- Footer -->
<footer id="footer">
    <div class="container">
        <div class="row double">
            <div class="6u">
                <div class="row collapse-at-2">
                    <div class="6u">
			<section class="special">
				<a href="http://www.tsl.ac.uk" class="image fit"><img src="http://localhost:4000/images/TSL_RGB.png" alt="" /></a>
			</section>
                    </div>
                    <div class="6u">
			    <h3><a href="http://www.tsl.ac.uk/">The Sainsbury Laboratory</a></h3>
                        <ul class="alt">
                            <li><a href="http://www.tsl.ac.uk/groups/moscou-group/">Moscou Group at TSL</a></li>
                            <li><a href="http://www.tsl.ac.uk/about-tsl/">About TSL</a></li>
                        </ul>
                    </div>
                </div>
            </div>
            <div class="6u">
                <h2>Moscou Group at The Sainsbury Laboratory</h2>
                <p>The Moscou group focuses on understanding immunity in the grasses. Current themes in our research include dissecting the genetic architecture of nonhost resistance, tradeoffs between biotrophs and necrotrophs, seedling versus adult plant resistance, and suppression of resistance in polyploid genomes. The group uses genetics and bioinformatics to explore biology and solve long-standing puzzles.</p>
		<p>The views on this website are those of its author.</p>
                <ul class="icons">
                    <li><a href="https://twitter.com/matthewmoscou?lang=en" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
                    <li><a href="https://github.com/matthewmoscou" class="icon fa-github"><span class="label">Github</span></a></li>
                </ul>
            </div>
        </div>
        <ul class="copyright">
            <li>&copy; Matthew Moscou. All rights reserved.</li>
	    <li>Layout <a href="https://github.com/old-jekyll-templates/Ion-Jekyll-Theme">Ion Theme</a></li>
        </ul>
    </div>
</footer>


    </body>

</html>
