---
layout: nosidebar
title: CV
permalink: /CV/
---

## <b>Matthew James Moscou</b>

![alt text](../images/orcid-logo.png "ORCID ID"){:height="24px" width="24px"} [0000-0003-2098-6818](http://orcid.org/0000-0003-2098-6818)   
![alt text](../images/rid_logo.gif "ORCID ID"){:height="15px" width="130px"} [D-5266-2011](http://www.researcherid.com/rid/D-5266-2011)

## Address
The Sainsbury Laboratory  
Norwich Research Park  
Norwich  
NR4 7UH  
United Kingdom

## Contact information
Office: +44 (0) 1603 450 296  
Email: matthew.moscou@sainsbury-laboratory.ac.uk

## Education
Iowa State University, Bioinformatics and Computational Biology, Ph.D., 2010   
University of California, Riverside, Pure Mathematics, B.S., 2004   
University of California, Riverside, Physics, B.S., 2004   

## Dissertation
Deciphering the transcriptional regulation and response of barley to obligate fungal biotroph invasion  
Advisors: Roger Wise and Julie Dickerson

## Professional Experience
2014-present		Group Leader, The Sainsbury Laboratory, UK  
2013-2014		Team Leader, 2Blades Group, The Sainsbury Laboratory, UK  
2010-2012	Post-doctoral Research Associate, 2Blades Group, The Sainsbury Laboratory, UK  
2005-2010		Graduate Research Assistant, Iowa State University  
Summer 2009	Summer Intern, Syngenta Seeds, Inc. – Statistical Genetics Research Group  
2004-2005		Programmer/Analyst I, University of California, Riverside  

## Teaching, Mentoring, and Supervision
Supervisor of four Ph.D. and four Msc students (TSL, 2011-present)  
Lecturer at the University of East Anglia for BIO-03C46 (Plant Biotechnology for Sustainable Food Production) (UEA, 2015)  
EMBO Practical Course on Plant-Microbe Interactions, Lecturer (TSL, 2012)  
Molecular Biology of Plant-Pathogen Interaction, Guest Lecturer (ISU, 2008)  
Computational and Systems Biology Summer Institute, Lecturer/Teaching Assistant (ISU, 2007)  
National Science Foundation, Research Experience for Teachers, Mentor (ISU, 2006)  
Bioinformatics and Computational Biology Summer Institute, Lecturer/Teaching Assistant (ISU, 2006)  

## Fellowships and Awards
[Human Frontier Science Program Fellow](www.hfsp.org), Long-Term Fellowship, 2011-2014  
Graduate Research Excellence Award, [Iowa State University](www.iastate.edu), 2010  
[BioVision.Nxt](http://www.biovision.org) Fellow, 2009  
James Cornette Research Fellowship in Bioinformatics and Computational Biology, [Iowa State University](www.iastate.edu), 2008  

## Publications
### 2017
Yu, G., Champouret, N., Steuernagel, B., Olivera, P.D., Simmons, J. Williams, C. Johnson, R., Moscou, M.J., Hernández-Pinzón, I., Green, P., Sela, H., Millet, E., Jones, J.D.G., Ward, E.R., Steffenson, B.J., and Wulff, B.B.H. (2017) Discovery and characterization of two new stem rust resistance genes in <i>Aegilops sharonensis</i>. <b><i>Theoretical and Applied Genetics</i></b> Accepted. [doi:10.1007/s00122-017-2882-8](https://doi.org/10.1007/s00122-017-2882-8)

Hisano, H., Meints, B., Moscou, M.J., Cistue, L., Echávarri, B., Sato, K., and Hayes, P.M. (2017) Selection of transformation-efficient barley genotypes based on <i>TFA</i> (transformation amenability) haplotype and higher resolution mapping of the <i>TFA</i> loci. <b><i>Plant Cell Reports</i></b> <b>36</b>:611–620 [doi:10.1007/s00299-017-2107-2](https://doi.org/10.1007/s00299-017-2107-2)

Bettgenhaeuser, J., Corke, F.M.K., Opanowicz, M., Green, P., Hernández-Pinzón, I., Doonan, J.H., Moscou, M.J. (2017) Natural variation in <i>Brachypodium</i> links vernalization and flowering time loci as major flowering determinants. <b><i>Plant Physiology</i></b> <b>173</b>:256-268 [doi:10.1104/pp.16.00813](https://doi.org/10.1104/pp.16.00813)

### 2016
Le Fevre, R., O’Boyle, B., Moscou, M.J., and Schornack S. (2016) Colonization of barley by the broad-host hemibiotrophic pathogen <i>Phytophthora palmivora</i> uncovers a leaf development dependent involvement of <i>MLO</i>. <b><i>Molecular Plant-Microbe Interactions</i></b> <b>29</b>: 385-395 [doi:10.1094/MPMI-12-15-0276-R](https://doi.org/10.1094/MPMI-12-15-0276-R)

Dawson, A.M., Ferguson, J., Gardiner, M., Green, P., Hubbard, A., and Moscou, M.J. (2016) Isolation and fine mapping of <i>Rps6</i>: An intermediate host resistance gene in barley to wheat stripe rust. <b><i>Theoretical and Applied Genetics</i></b> <b>129</b>: 831-843 [doi:10.1007/s00122-015-2659-x](https://doi.org/10.1007/s00122-015-2659-x)

### 2015
Dawson, A.M., Bettgenhaeuser, J., Gardiner, M., Green, P., Hernández-Pinzón, I., Hubbard, A., and Moscou, M.J. (2015) The development of quick, robust, quantitative phenotypic assays for describing the host-nonhost landscape to stripe rust. <b><i>Frontiers in Plant Science</i></b> <b>6</b>: 876 [doi:10.3389/fpls.2015.00876](https://doi.org/10.3389/fpls.2015.00876)

Muñoz-Amatriaín, M., Lonardi, S., Luo, M. C., Madishetty, K., Svensson, J.T., Moscou, M.J., Wanamaker, S., Jiang, T., Kleinhofs, A., Muehlbauer, G.J., Wise, R., Stein, N., Ma, Y., Rodriguez, E., Kudrna, D., Bhat, P., Chao, S., Condamine, P., Heinen, S., Resnik, J., Wing, R., Witt, H., Alpert, M., Beccuti, M., Bozdag, S., Cordero, F., Mirebrahim, H., Ounit, R., Wu, Y., You, F., Zheng, J., Doležel, J., Grimwood, J., Schmutz, J., Duma, D., Altschmied, L., Blake, T., Bregitzer, P., Cooper, L., Dilbirligi, M., Falk, A., Feiz, L., Graner, A., Gustafson, P., Hayes, P.M., Lemaux, P., Mammadov, J., and Close, T.J. (2015) Sequencing of 15 622 gene-bearing BACs clarifies the gene-dense regions of the barley genome. <b><i>The Plant Journal</i></b> <b>84</b>: 216-227 [doi:10.1111/tpj.12959](https://doi.org/doi:10.1111/tpj.12959)

Knight, E., Binnie, A., Draeger, T., Moscou, M., Rey, M., Sucher, J., Mehra, S., King, I., and Moore, G. (2015) Mapping the ‘breaker’ element of the gametocidal locus proximal to a block of sub-telomeric heterochromatin on the long arm of chromosome 4S<sup>sh</sup> of <i>Aegilops sharonensis</i>. <b><i>Theoretical and Applied Genetics</i></b> <b>128</b>(6):1049-1059 [doi:10.1007/s00122-015-2489-x](https://doi.org/10.1007/s00122-015-2489-x)

### 2014
Wulff, B.H.H. and Moscou, M.J. (2014) Strategies for transferring resistance into wheat: from wide crosses to GM cassettes. <b><i>Frontiers in Plant Science</i></b> <b>5</b>:692 [doi:10.3389/fpls.2014.00664](https://doi.org/10.3389/fpls.2014.00664)

Bettgenhaeuser, J., Gilbert, M., Ayliffe, M., Moscou, M.J. (2014) Nonhost resistance to rust pathogens – a continuation of continua. <b><i>Frontiers in Plant Science</i></b> <b>5</b>:664 [doi:10.3389/fpls.2014.00664](https://doi.org/10.3389/fpls.2014.00664)

### 2013
Bouyioukos, C., Moscou, M.J., Champouret, N., Hernández-Pinzón I., Ward E.R. and Wulff B.B.H. (2013) Characterisation and analysis of the <i>Aegilops sharonensis</i> transcriptome, a wild relative of wheat in the Sitopsis section. <b><i>PLoS ONE</i><b> <b>8</b>(8): e72782. [doi:10.1371/journal.pone.0072782](https://doi.org/10.1371/journal.pone.0072782)

Ayliffe, M. Singh, D., Park, R., Moscou, M., Pryor, T. (2013) The infection of <i>Brachypodium distachyon</i> with selected grass rust pathogens. <b><i>Molecular Plant-Microbe Interactions</i></b> <b>26</b>(8): 946-957 [doi:10.1094/MPMI-01-13-0017-R](https://doi.org/10.1094/MPMI-01-13-0017-R)

Schornack, S., Moscou, M.J., Ward, E.R., and Horvath, D. M. (2013) Engineering plant disease resistance based on TAL effectors. <b><i>Annual Review of Phytopathology</i></b> <b>51</b>:383-406 [doi:10.1146/annurev-phyto-082712-102255](https://doi.org/10.1146/annurev-phyto-082712-102255)

### 2012
The International Barley Genome Sequencing Consortium (IBSC). (2012) A physical, genetic and functional sequence assembly of the barley genome. <b><i>Nature</i></b> <b>491</b>: 711-716 [doi:10.1038/nature11543](https://doi.org/10.1038/nature11543)

Moeller, J.R., Moscou, M.J., Bancroft, T., Skadsen, R.W., Wise, R.P., and Whitham, S.A. (2012) Differential accumulation of host mRNAs on polyribosomes during obligate pathogen-plant interactions. <b><i>Molecular BioSystems</i></b> <b>8</b>:2153-2165 [doi:10.1039/C2MB25014D](https://doi.org/10.1039/C2MB25014D)

### 2011
Muñoz-Amatriaín, M.<sup>![alt text](../images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Moscou, M.J.<sup>![alt text](../images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Bhat, P.R., Svensson, J.T., Bartoš, J., Suchánková, P., Šimková, H., Endo, T.R., Fenton, R. D., Wu, Y., Lonardi, S., Castillo, A.M., Chao, S., Cistué, L., Cuesta-Marcos, A., Forrest, K., Hayden, M.J., Hayes, P.M., Horsley, R.D., Kleinhofs, A., Moody, D., Sato, K., Vallés, M.P., Wulff, B.B.H., Muehlbauer, G.J., Doležel, J., and Close, T.J. (2011) An improved consensus linkage map of barley based on flow-sorted chromosomes and SNP markers. <b><i>The Plant Genome</i></b> <b>4</b>(3): 238-249 [doi:10.3835/plantgenome2011.08.0023](https://doi.org/10.3835/plantgenome2011.08.0023)

Moscou, M.J., Lauter, N., Steffenson, B., and Wise, R.P. (2011) Quantitative and qualitative stem rust resistance factors in barley are associated with transcriptional suppression of defense regulons. <b><i>PLoS Genetics</i></b> <b>7</b>(7): e1002208 [doi:10.1371/journal.pgen.1002208](https://doi.org/10.1371/journal.pgen.1002208)

Moscou, M.J., Lauter, N., Caldo, R.A., Nettleton, D., and Wise, R.P. (2011) Quantitative and temporal definition of the <i>Mla</i> transcriptional regulon during barley-powdery mildew interactions.  <b><i>Molecular Plant-Microbe Interactions</i></b> <b>24</b>(6): 694-705 [doi:10.1094/MPMI-09-10-0211](https://doi.org/10.1094/MPMI-09-10-0211)

### 2009
Close, T.J., Bhat, P.R., Lonardi, S., Wu, Y., Rostoks, N., Ramsay, L., Druka, A., Stein, N., Svensson, J.T., Wanamaker, S., Bozdag, S., Roose, M.L., Moscou, M.J., Chao, S., Varshney, R., Szűcs, P., Sato, K., Hayes, P.M., Matthews, D.E., Kleinhofs, A., Muehlbauer, G. J., DeYoung, J., Marshall, D.F., Madishetty, K., Fenton, R.D., Condamine, P., Graner, A., Waugh, R. 2009. Development and implementation of high-throughput SNP genotyping in barley. <b><i>BMC Genomics</i></b> <b>10</b>: 582 [doi:10.1186/1471-2164-10-582](https://doi.org/10.1186/1471-2164-10-582)

Moscou, M.J. and Bogdanove, A. (2009) A simple cipher governs DNA recognition by TAL effectors. <b><i>Science</i></b> <b>326</b>: 1501 [doi:10.1126/science.1178817](https://doi.org/10.1126/science.1178817)

Xi, L., Moscou, M.J., Meng, Y., Xu, W. Caldo, R.A., Shaver, M., Nettleton, D., Wise, R.P. (2009) Transcript-based cloning of <i>HvRRP46</i>, a regulator of rRNA processing and <i>R</i> gene-independent cell death in barley-powdery mildew interactions. <b><i>Plant Cell</i></b> <b>21</b>: 3280-3295 [doi:10.1105/tpc.109.066167](https://doi.org/10.1105/tpc.109.066167)

Meng, Y.<sup>![alt text](../images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Moscou, M.J.<sup>![alt text](../images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Wise, R.P. (2009) <i>Blufensin1</i> negatively impacts basal defense in response to barley powdery mildew. <b><i>Plant Physiology</i></b> <b>149</b>(1): 271-285 [doi:10.1104/pp.108.129031](https://doi.org/10.1104/pp.108.129031)

### 2008
Lauter, N., Moscou, M.J., Habinger, J., Moose, S. (2008) Quantitative genetic dissection of shoot architecture traits in maize: Towards a functional genomics approach. <b><i>The Plant Genome</i></b> <b>1</b>(2): 99-110 [doi:10.3835/plantgenome2008.06.0385](https://doi.org/10.3835/plantgenome2008.06.0385)

Potokina, E., Druka, A., Luo, Z., Moscou, M., Wise, R., Waugh, R., Kearsey, M. (2008) Tissue-dependent limited pleiotropy affects gene expression in barley. <b><i>Plant Journal</i></b> <b>56</b>(2): 287-296 [doi:10.1111/j.1365-313X.2008.03601.x](https://doi.org/10.1111/j.1365-313X.2008.03601.x)

### 2007
Wise, R.P., Moscou, M.J., Bogdanove, A.J., Whitham, S.A. (2007) Transcript profiling in host–pathogen interactions. <b><i>Annual Review of Phytopathology</i></b> <b>45</b>: 329-369 [doi:10.1146/annurev.phyto.45.011107.143944](https://doi.org/10.1146/annurev.phyto.45.011107.143944)

## Preprints
Clare, S., Kitcher, W., Gardiner, M., Green, P., Hubbard, A., and Moscou, M.J. (2016) Defining the genetic architecture of stripe rust resistance in the barley accession HOR 1428. <b><i>bioRxiv</i></b> 093773 [doi:10.1101/093773](https://doi.org/10.1101/093773)

## Conference Publications
Champouret N, Moscou MJ, Bouyioukos C, Steuernagel B, Hernández-Pinzón I, Green P, Kaufman J, Olivera PD, Pretorius Z, Millet E, Steffenson BJ, Ward ER and Wulff BBH (2011) A pipeline for cloning resistance genes effective against African stem rust races from the diploid wheat relative Aegilops sharonensis In: Proc Borlaug Global Rust Initiative, June 13–19. McIntosh, R. (ed). pp. 66–74

Wise, R. P., Meng, Y., Moscou, M. J., Xu, W. 2009. Regulators of innate immunity in cereal-fungal interactions. In Biology of Plant Microbe Interactions, Volume 7. International Society for Molecular Plant-Microbe Interactions, St. Paul, MN

Moscou, M.J., Caldo, R.A., Lauter, N., Wise, R.P. 2008. Construction of coexpression networks to explore barley-powdery mildew interactions. Paper 113 in: Biology of Plant Microbe Interactions, Volume 6. M. Lorito, S. L. Woo, and F. Scala, eds. International Society for Molecular Plant-Microbe Interactions, St. Paul, MN

## Invited Talks
Dissecting the genetic architecture of nonhost resistance to stripe rust. Institut de Recherche pour le Développement. Montpellier, France. April 21, 2017

The process of discovery: How a conversation at a conference led to the development of a research program. Plant Sciences Graduate Student Symposium. Saskatoon, Saskatchewan, Canada. April 1, 2017

Dissecting the genetic architecture of nonhost resistance to stripe rust. University of Saskatchewan. Saskatoon, Saskatchewan, Canada. March 30, 2017

Accessing natural variation in <i>Brachypodium distachyon</i>: Development of the ABR6 x Bd21 RIL population. Plant and Animal Genome Conference. San Diego, CA, USA. January 17, 2017

Genomics of the barley immune system. Plant and Animal Genome Conference. San Diego, CA, USA. January 14, 2017

Engineering durable disease resistance. Gatsby Plant Science Network Meeting. Oxford, United Kingdom. September 9, 2016

Dual specificity at the <i>Mla</i> locus confers resistance to barley powdery mildew and wheat stripe rust. International Powdery Mildew Symposium. Portland, Oregon, USA. July 16, 2016

The barley ‘nibblerome’: Defining the set of NB-LRR-type <i>R</i> genes from a diverse collection of barley. International Barley Genetics Symposium. Minneapolis, Minnesota, USA. June 28, 2016

Map-based cloning of host and nonhost resistance genes in barley. USDA-ARS, Cereal Crops Research Unit, North Dakota State University, Fargo, North Dakota, USA, December 1, 2015

Dissecting the genetic architecture of nonhost resistance to stripe rust. Plant Biotic Stresses & Resistance Mechanisms II, Vienna, Austria. July 3, 2015

Dissecting the basis of host species specificity and nonhost resistance to wheat stripe rust. Heinrich-Heine-Universität Düsseldorf, Düsseldorf, Germany. November 24, 2014

Nonhost resistance as a potential source for controlling wheat rust disease. International Congress on Molecular Plant-Microbe Interactions, Rhodes, Greece. July 10, 2014

Harnessing the loci that determine the inappropriate host status of barley to engineer durable resistance to wheat stripe rust. North American Barley Researcher Workshop, Minneapolis, Minnesota, USA. July 2, 2014

Why does wheat stripe rust not infect barley? Plant and Animal Genome Conference, San Diego, California, USA. January 11, 2014

Deciphering the genetic architecture of host species specificity and nonhost resistance to wheat stripe rust. Oregon State University, Corvallis, Oregon, USA. January 8, 2014

The genetic architecture underlying host species specificity of stripe rust. Washington State University, Pullman, Washington, USA. January 6, 2014

Harnessing the loci underlying host species specificity to engineer durable resistance to wheat stripe rust. Keystone Symposia: Plant Immunity: Pathways and Translation. Big Sky, Montana, USA. April 7th-April 12th, 2013

Host species specificity to stripe rust in barley colocalizes with loci conditioning resistance to powdery mildew. University of Minnesota, St. Paul, Minnesota, USA. March 28th, 2013

<i>spst1</i>, a novel susceptibility gene in barley to wheat stripe rust. 13th International Cereal Rust and Powdery Mildew Conference. Beijing, China. August 28th-September 1st, 2012

Microscopy-based quantitative analysis of nonhost resistance in Brachypodium distachyon to cereal rusts. Monogram Conference. Aberystwyth, United Kingdom. March 26th-28th, 2012

Epistatic control of powdery mildew-induced necrosis among progeny of the Steptoe x Morex doubled-haploid mapping population in barley.  John Innes Centre (JIC), Norwich, United Kingdom. March 17, 2009

Natural genetic variation in barley identifies master regulatory loci of defense-related genes. Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Gatersleben, Germany. March 19, 2008

Gene-for-gene-mediated transcriptome reprogramming in barley-powdery mildew interactions. Plant and Animal Genome Conference, San Diego, California, USA. January 12, 2008

Bioinformatics and Computational Biology Laboratory: An interaction model for cross-departmental bioinformatics knowledge integration (http://lab.bcb.iastate.edu). New Mexico State University, Las Cruces, New Mexico, USA. November 14, 2007

PLEXdb: Plant and pathogen expression database and tools for comparative and functional genomics analysis. COST Action 858, INRA, Bordeaux, France. November 20, 2006

Markermaker: an iterative algorithm for optimization of fit between expression level polymorphisms and genetic marker datasets. Sixth Annual Joint Bioinformatics Symposium, Iowa State University, Ames, Iowa, USA. July 14, 2006

## Peer Review
Manuscripts reviewed for BMC Genomics, BMC Plant Biology, Frontiers in Plant Science, Genome Biology, Journal of Experimental Botany, Molecular Genetics and Genomics, Molecular Plant Pathology, Nature Communications, Nature Genetics, Physiological and Molecular Plant Pathology, Phytopathology, Plant Cell Reports, Plant Health Progress, Plant Management Network, PLoS Genetics, PLoS ONE, Rice, Science, and Theoretical and Applied Genetics.

![alt text](../images/publons.png "Publons"){:height="24px" width="24px"} [https://publons.com/a/602996/](https://publons.com/a/602996/)

## Membership in Professional Societies
American Association for the Advancement of Science  
American Society of Plant Biologists  
British Society of Plant Pathology  
Genetics Society of America  
International Society of Molecular Plant-Microbe Interactions  


Last updated: 2017 April 3 
