---
layout: nosidebar
title: Publications
permalink: /Publications/
---

## Preprints at [bioRxiv](https://www.biorxiv.org/)
Brabham, H. J., Hernández-Pinzón, I., Holden, S., Lorang, J., and Moscou, M.J. (2017) An ancient integration in a plant NLR is maintained as a <i>trans</i>-species polymorphism. <b><i>bioRxiv</i></b> 239541 [doi:10.1101/239541](http://doi.org/10.1101/239541)
  * [Raw data for figures and <i>de novo</i> assembled transcriptomes on figshare](https://figshare.com/account/home#/projects/27682)
  * [Phylogenetic trees on iTOL](http://itol.embl.de/) - Under 'Sharing data' use the iTOL login: moscou
  * [Scripts and Exo70 annotation](https://github.com/matthewmoscou/Exo70)
  * [Raw sequencing and assembled genomic and transcriptomic data](https://www.ncbi.nlm.nih.gov)
   1. [Barley RNAseq set 1](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA378334/)
   2. [Barley RNAseq set 2](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA378723/)
   3. [Oat RNAseq and RenSeq/PacBio](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA422803/)
   4. [Barley RenSeq/PacBio](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA422986/)

Clare, S., Kitcher, W., Gardiner, M., Green, P., Hubbard, A., and Moscou, M.J. (2016) Defining the genetic architecture of stripe rust resistance in the barley accession HOR 1428. <b><i>bioRxiv</i></b> 093773 [doi:10.1101/093773](https://doi.org/10.1101/093773)

## 2020
Panstruga, R. and Moscou, M.J. (2020) What is the molecular basis of nonhost resistance? <b><i>Molecular Plant-Microbe Interactions</i></b> [doi:10.1094/](https://doi.org/10.1094/)

Read, A.C.<sup>![alt text](../images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Hutin, M.<sup>![alt text](../images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Moscou, M.J., Rinaldi, F.C., and Bogdanove, A. J. (2020) Cloning of the rice <i>Xo1</i> resistance gene and interaction of the Xo1 protein with the defense-suppressing <i>Xanthomonas</i> effector Tal2h <b><i>Molecular Plant-Microbe Interactions</i></b> [doi:10.1094/MPMI-05-20-0131-SC](https://doi.org/10.1094/MPMI-05-20-0131-SC)

Hiebert, C.W.<sup>![alt text](https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Moscou, M.J.<sup>![alt text](https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Hewitt, T., Steuernagel, B., Hernández-Pinzón, I., Green, P., Pujol, V., Zhang, P., Rouse, M.N., Jin, Y., McIntosh, R.A., Upadhyaya, N., Zhang, J., Bhavani, S., Vrána, J., Karafiátová, M., Huang, L., Fetch, T., Doležel, J., Wulff, B.B.H., Lagudah, E., Spielmeyer, W. (2020) Stem rust resistance in wheat is suppressed by a subunit of the Mediator complex <b><i>Nature Communications</i></b> [doi:10.1038/s41467-020-14937-2](https://doi.org/10.1038/s41467-020-14937-2)  
  * [Raw data used to generate figures](https://figshare.com/projects/Stem_rust_resistance_in_wheat_is_suppressed_by_a_subunit_of_the_Mediator_complex/28056)
  * [Scripts from the Moscou group](https://github.com/matthewmoscou/Canthatch)  
  * [Scripts from the Lagudah group](https://github.com/TC-Hewitt/MuTrigo)  
  * [Raw sequencing and assembled genomic and transcriptomic data](https://www.ncbi.nlm.nih.gov)
   1. [Chromosome flow sorting gDNA and RNAseq](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA401266)
   2. [<i>De novo</i> assembly of Canthatch](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23265)

Wei, B., Moscou, M.J., Sato, K., Gourlie, R., Strelkov, S., Aboukhaddour, R. (2020) Identification of a locus conferring dominant susceptibility to *Pyrenophora tritici-repentis* in barley. <b><i>Frontiers in Plant Science</i></b> [doi:10.3389/fpls.2020.00158](https://doi.org/10.3389/fpls.2020.00158)

## 2018
Bettgenhaeuser, J., Gardiner, M., Spanner, R., Green, P., Hernández-Pinzón, I., Hubbard, A., Ayliffe, M., and Moscou, M.J. (2018) The genetic architecture of colonization resistance in <i>Brachypodium distachyon</i> to non-adapted stripe rust (<i>Puccinia striiformis</i>) isolates. <b><i>PLoS Genetics</i></b> <b>14</b>(9):e1007637. [doi:10.1371/journal.pgen.1007637](https://doi.org/10.1371/journal.pgen.1007637)

Gilbert, B., Bettgenhaeuser, J., Upadhyaya, N., Soliveres, M., Singh, D., Park, R.F., Moscou, M.J., and Ayliffe, M. (2018) Components of <i>Brachypodium distachyon</i> resistance to the nonadapted wheat stripe rust pathogens are simply inherited. <b><i>PLoS Genetics</i></b> <b>14</b>(9):e1007636. [doi:10.1371/journal.pgen.1007636](https://doi.org/10.1371/journal.pgen.1007636)

Ghosh, S., Watson, A., Gonzalez-Navarro, O.E., Ramirez-Gonzalez, R.H., Yanes, L., Mendoza-Suarez, M., Simmonds, J., Wells, R., Rayner, T., Green, P., Hafeez, A., Hayta, S., Melton, R.E., Steed, A., Sarkar, A., Carter, J., Perkins, L., Lord, J., Tester, M., Osbourn, A., Moscou, M.J., Nicholson, P., Harwood, W., Martin, C., Domoney, C., Uauy, C., Hazard, B., Wulff, B.B.H., Hickey, L.T. (2018) Speed breeding in growth chambers and glasshouses for crop breeding and model plant research. <b><i>Nature Protocols</i></b> <b>13</b>:2944-2963. [doi:10.1038/s41596-018-0072-z](https://doi.org/10.1038/s41596-018-0072-z)

Omidvar, V., Dugyala, S., Li, F., Rottschaefer, S., Miller, M.E., Ayliffe, M., Moscou, M.J., Kianian, S.F., Figueroa, M. (2018) Detection of race-specific resistance against <i>Puccinia coronata</i> f. sp. <i>avenae</i> in <i>Brachypodium</i> species. <b><i>Phytopathology</i></b> <b>108</b>:1443-1454. [doi:10.1094/PHYTO-03-18-0084-R](https://doi.org/10.1094/PHYTO-03-18-0084-R)

Bailey, P.C., Schudoma, C., Jackson, W., Baggs, E., Dagdas, G., Haerty, W., Moscou, M., Krasileva, K.V. (2018) Dominant integration locus drives continuous diversification of plant immune receptors with exogenous domain fusions. <b><i>Genome Biology</i></b> <b>19</b>:23. [doi:10.1186/s13059-018-1392-6](https://doi.org/10.1186/s13059-018-1392-6)
  * [Phylogenetic trees on iTOL](http://itol.embl.de/) - Under 'Sharing data' use the iTOL login: KrasilevaGroup
  * [Scripts for NLR-ID identification](https://github.com/krasileva-group/plant_rgenes)
  * [Scripts for tandem NLR analysis](https://github.com/krasileva-group/tandem)
  * [Scripts for CID domain analysis](https://github.com/matthewmoscou/NLR-ID_motif)

## 2017
Moscou, M. J. and van Esse, H. P. (2017) The quest for durable resistance. <b><i>Science</i></b> <b>358</b>:1541-1542. [doi:10.1126/science.aar4797](http://science.sciencemag.org/cgi/content/full/358/6370/1541?ijkey=/LGSoXCyQFClw&keytype=ref&siteid=sci)

Herb, D.W., Benson, A., Carey, D., Cistue, L., Filichkin, T, Fisk, S.P., Helgerson, L., Jennings, R., Li, Y., Meints, B.M., Monsour, R., Moscou, M., Nguygen, A., Onio, A., Romagosa, I., Thiel, R., Thomas, W.B., Tynan, S.P., Vega, V., Vinkemeier, K., Hayes, P.M. (2017) Malt modification and its effects on the contributions of barley genotype to beer flavor. <b><i>Journal of American Society of Brewing Chemists</i></b>. <b>4</b>:354-362 [doi:10.1094/ASBCJ-2017-4976-01](http://doi.org/10.1094/ASBCJ-2017-4976-01)

Herb, D.W., Meints, B.M., Jennings, R., Romagosa, I., Moscou, M., Carey, D., Cistue, L., Filichkin, T, Fisk, S.P., Helgerson, L., Martens, C., Monsour, R., Thiel, R., Tynan, S., Thomas, W.T.B., Vinkemeier, K., Hayes, P.M. 2017. Effects of barley (<i>Hordeum vulgare</i> L.) variety and growing environment on beer flavor. Journal of American Society of Brewing Chemists. <b>4</b>:345-353 [doi:10.1094/ASBCJ-2017-4860-01](http://doi.org/10.1094/ASBCJ-2017-4860-01)

Yu, G., Champouret, N., Steuernagel, B., Olivera, P.D., Simmons, J. Williams, C. Johnson, R., Moscou, M.J., Hernández-Pinzón, I., Green, P., Sela, H., Millet, E., Jones, J.D.G., Ward, E.R., Steffenson, B.J., and Wulff, B.B.H. (2017) Discovery and characterization of two new stem rust resistance genes in <i>Aegilops sharonensis</i>. <b><i>Theoretical and Applied Genetics</i></b> <b>130</b>:1207–1222 [doi:10.1007/s00122-017-2882-8](https://doi.org/10.1007/s00122-017-2882-8)

Hisano, H., Meints, B., Moscou, M.J., Cistue, L., Echávarri, B., Sato, K., and Hayes, P.M. (2017) Selection of transformation-efficient barley genotypes based on <i>TFA</i> (transformation amenability) haplotype and higher resolution mapping of the <i>TFA</i> loci <b><i>Plant Cell Reports</i></b> <b>36</b>:611–620 [doi:10.1007/s00299-017-2107-2](https://doi.org/10.1007/s00299-017-2107-2)

Bettgenhaeuser, J., Corke, F.M.K., Opanowicz, M., Green, P., Hernández-Pinzón, I., Doonan, J.H., Moscou, M.J. (2017) Natural variation in <i>Brachypodium</i> links vernalization and flowering time loci as major flowering determinants. <b><i>Plant Physiology</i></b> <b>173</b>:256-268 [doi:10.1104/pp.16.00813](https://doi.org/10.1104/pp.16.00813)

## 2016
Le Fevre, R., O’Boyle, B., Moscou, M.J., and Schornack S. (2016) Colonization of barley by the broad-host hemibiotrophic pathogen <i>Phytophthora palmivora</i> uncovers a leaf development dependent involvement of <i>MLO</i>. <b><i>Molecular Plant-Microbe Interactions</i></b> <b>29</b>: 385-395 [doi:10.1094/MPMI-12-15-0276-R](https://doi.org/10.1094/MPMI-12-15-0276-R)

Dawson, A.M., Ferguson, J., Gardiner, M., Green, P., Hubbard, A., and Moscou, M.J. (2016) Isolation and fine mapping of <i>Rps6</i>: An intermediate host resistance gene in barley to wheat stripe rust. <b><i>Theoretical and Applied Genetics</i></b> <b>129</b>: 831-843 [doi:10.1007/s00122-015-2659-x](https://doi.org/10.1007/s00122-015-2659-x)

## 2015
Dawson, A.M., Bettgenhaeuser, J., Gardiner, M., Green, P., Hernández-Pinzón, I., Hubbard, A., and Moscou, M.J. (2015) The development of quick, robust, quantitative phenotypic assays for describing the host-nonhost landscape to stripe rust. <b><i>Frontiers in Plant Science</i></b> <b>6</b>: 876 [doi:10.3389/fpls.2015.00876](https://doi.org/10.3389/fpls.2015.00876)

Muñoz-Amatriaín, M., Lonardi, S., Luo, M. C., Madishetty, K., Svensson, J.T., Moscou, M.J., Wanamaker, S., Jiang, T., Kleinhofs, A., Muehlbauer, G.J., Wise, R., Stein, N., Ma, Y., Rodriguez, E., Kudrna, D., Bhat, P., Chao, S., Condamine, P., Heinen, S., Resnik, J., Wing, R., Witt, H., Alpert, M., Beccuti, M., Bozdag, S., Cordero, F., Mirebrahim, H., Ounit, R., Wu, Y., You, F., Zheng, J., Doležel, J., Grimwood, J., Schmutz, J., Duma, D., Altschmied, L., Blake, T., Bregitzer, P., Cooper, L., Dilbirligi, M., Falk, A., Feiz, L., Graner, A., Gustafson, P., Hayes, P.M., Lemaux, P., Mammadov, J., and Close, T.J. (2015) Sequencing of 15 622 gene-bearing BACs clarifies the gene-dense regions of the barley genome. <b><i>The Plant Journal</i></b> <b>84</b>: 216-227. [doi:10.1111/tpj.12959](https://doi.org/doi:10.1111/tpj.12959)

Knight, E., Binnie, A., Draeger, T., Moscou, M., Rey, M., Sucher, J., Mehra, S., King, I., and Moore, G. (2015) Mapping the ‘breaker’ element of the gametocidal locus proximal to a block of sub-telomeric heterochromatin on the long arm of chromosome 4S<sup>sh</sup> of <i>Aegilops sharonensis</i>. <b><i>Theoretical and Applied Genetics</i></b> <b>128</b>(6):1049-1059 [doi:10.1007/s00122-015-2489-x](https://doi.org/10.1007/s00122-015-2489-x)

## 2014
Wulff, B.H.H. and Moscou, M.J. (2014) Strategies for transferring resistance into wheat: from wide crosses to GM cassettes. <b><i>Frontiers in Plant Science</i></b> <b>5</b>:692 [doi:10.3389/fpls.2014.00664](https://doi.org/10.3389/fpls.2014.00664)

Bettgenhaeuser, J., Gilbert, M., Ayliffe, M., Moscou, M.J. (2014) Nonhost resistance to rust pathogens – a continuation of continua. <b><i>Frontiers in Plant Science</i></b> <b>5</b>:664 [doi:10.3389/fpls.2014.00664](https://doi.org/10.3389/fpls.2014.00664)

## 2013
Bouyioukos, C., Moscou, M.J., Champouret, N., Hernández-Pinzón I., Ward E.R. and Wulff B.B.H. (2013) Characterisation and analysis of the <i>Aegilops sharonensis</i> transcriptome, a wild relative of wheat in the Sitopsis section. <b><i>PLoS ONE</i><b> <b>8</b>(8): e72782. [doi:10.1371/journal.pone.0072782](https://doi.org/10.1371/journal.pone.0072782)

Ayliffe, M. Singh, D., Park, R., Moscou, M., Pryor, T. (2013) The infection of <i>Brachypodium distachyon</i> with selected grass rust pathogens. <b><i>Molecular Plant-Microbe Interactions</i></b> <b>26</b>(8): 946-957 [doi:10.1094/MPMI-01-13-0017-R](https://doi.org/10.1094/MPMI-01-13-0017-R)

Schornack, S., Moscou, M.J., Ward, E.R., and Horvath, D. M. (2013) Engineering plant disease resistance based on TAL effectors. <b><i>Annual Review of Phytopathology</i></b> <b>51</b>:383-406 [doi:10.1146/annurev-phyto-082712-102255](https://doi.org/10.1146/annurev-phyto-082712-102255)

## 2012
The International Barley Genome Sequencing Consortium (IBSC). (2012) A physical, genetic and functional sequence assembly of the barley genome. <b><i>Nature</i></b> <b>491</b>: 711-716 [doi:10.1038/nature11543](https://doi.org/10.1038/nature11543)

Moeller, J.R., Moscou, M.J., Bancroft, T., Skadsen, R.W., Wise, R.P., and Whitham, S.A. (2012) Differential accumulation of host mRNAs on polyribosomes during obligate pathogen-plant interactions. <b><i>Molecular BioSystems</i></b> <b>8</b>:2153-2165 [doi:10.1039/C2MB25014D](https://doi.org/10.1039/C2MB25014D)

## 2011
Muñoz-Amatriaín, M.<sup>![alt text](https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Moscou, M.J.<sup>![alt text](https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Bhat, P.R., Svensson, J.T., Bartoš, J., Suchánková, P., Šimková, H., Endo, T.R., Fenton, R. D., Wu, Y., Lonardi, S., Castillo, A.M., Chao, S., Cistué, L., Cuesta-Marcos, A., Forrest, K., Hayden, M.J., Hayes, P.M., Horsley, R.D., Kleinhofs, A., Moody, D., Sato, K., Vallés, M.P., Wulff, B.B.H., Muehlbauer, G.J., Doležel, J., and Close, T.J. (2011) An improved consensus linkage map of barley based on flow-sorted chromosomes and SNP markers. <b><i>The Plant Genome</i></b> <b>4</b>(3): 238-249 [doi:10.3835/plantgenome2011.08.0023](https://doi.org/10.3835/plantgenome2011.08.0023)

Moscou, M.J., Lauter, N., Steffenson, B., and Wise, R.P. (2011) Quantitative and qualitative stem rust resistance factors in barley are associated with transcriptional suppression of defense regulons. <b><i>PLoS Genetics</i></b> <b>7</b>(7): e1002208 [doi:10.1371/journal.pgen.1002208](https://doi.org/10.1371/journal.pgen.1002208)

Moscou, M.J., Lauter, N., Caldo, R.A., Nettleton, D., and Wise, R.P. (2011) Quantitative and temporal definition of the <i>Mla</i> transcriptional regulon during barley-powdery mildew interactions.  <b><i>Molecular Plant-Microbe Interactions</i></b> <b>24</b>(6): 694-705 [doi:10.1094/MPMI-09-10-0211](https://doi.org/10.1094/MPMI-09-10-0211)

## 2009
Close, T.J., Bhat, P.R., Lonardi, S., Wu, Y., Rostoks, N., Ramsay, L., Druka, A., Stein, N., Svensson, J.T., Wanamaker, S., Bozdag, S., Roose, M.L., Moscou, M.J., Chao, S., Varshney, R., Szűcs, P., Sato, K., Hayes, P.M., Matthews, D.E., Kleinhofs, A., Muehlbauer, G. J., DeYoung, J., Marshall, D.F., Madishetty, K., Fenton, R.D., Condamine, P., Graner, A., Waugh, R. 2009. Development and implementation of high-throughput SNP genotyping in barley. <b><i>BMC Genomics</i></b> <b>10</b>: 582 [doi:10.1186/1471-2164-10-582](https://doi.org/10.1186/1471-2164-10-582)

Moscou, M.J. and Bogdanove, A. (2009) A simple cipher governs DNA recognition by TAL effectors. <b><i>Science</i></b> <b>326</b>: 1501 [doi:10.1126/science.1178817](https://doi.org/10.1126/science.1178817)

Xi, L., Moscou, M.J., Meng, Y., Xu, W. Caldo, R.A., Shaver, M., Nettleton, D., Wise, R.P. (2009) Transcript-based cloning of <i>HvRRP46</i>, a regulator of rRNA processing and <i>R</i> gene-independent cell death in barley-powdery mildew interactions. <b><i>Plant Cell</i></b> <b>21</b>: 3280-3295 [doi:10.1105/tpc.109.066167](https://doi.org/10.1105/tpc.109.066167)

Meng, Y.<sup>![alt text](https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Moscou, M.J.<sup>![alt text](https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/images/yin_yang.png "Authors contributed equally"){:height="12px" width="12px"}</sup>, Wise, R.P. (2009) <i>Blufensin1</i> negatively impacts basal defense in response to barley powdery mildew. <b><i>Plant Physiology</i></b> <b>149</b>(1): 271-285 [doi:10.1104/pp.108.129031](https://doi.org/10.1104/pp.108.129031)

## 2008
Lauter, N., Moscou, M.J., Habinger, J., Moose, S. (2008) Quantitative genetic dissection of shoot architecture traits in maize: Towards a functional genomics approach. <b><i>The Plant Genome</i></b> <b>1</b>(2): 99-110 [doi:10.3835/plantgenome2008.06.0385](https://doi.org/10.3835/plantgenome2008.06.0385)

Potokina, E., Druka, A., Luo, Z., Moscou, M., Wise, R., Waugh, R., Kearsey, M. (2008) Tissue-dependent limited pleiotropy affects gene expression in barley. <b><i>Plant Journal</i></b> <b>56</b>(2): 287-296 [doi:10.1111/j.1365-313X.2008.03601.x](https://doi.org/10.1111/j.1365-313X.2008.03601.x)

## 2007
Wise, R.P., Moscou, M.J., Bogdanove, A.J., Whitham, S.A. (2007) Transcript profiling in host–pathogen interactions. <b><i>Annual Review of Phytopathology</i></b> <b>45</b>: 329-369 [doi:10.1146/annurev.phyto.45.011107.143944](https://doi.org/10.1146/annurev.phyto.45.011107.143944)


Last updated: 2020 November 30
