---
layout: research
title: Research
permalink: /Research/
---

## Genetic architecture of nonhost resistance in grasses to fungal pathogens
Collaborations: Michael Ayliffe at [CSIRO, Black Mountain](https://www.csiro.au/en/Locations/ACT/Black-Mountain), Wolfgang Spielmeyer at [CSIRO, Black Mountain](https://www.csiro.au/en/Locations/ACT/Black-Mountain), Zakkie Pretorius [University of the Free State](http://www.ufs.ac.za/natagri/departments-and-divisions/plant-sciences-home), and Melania Figueroa at [University of Minnesota](https://plpa.cfans.umn.edu/people/faculty/melania-figueroa)

Nonhost resistance has long been considered a source for engineering durable disease resistance. While we have an extensive mechanistic understanding of the genes underlying host resistance, little is known about the genes conditioning resistance in the transition from host to nonhost. We set out to understand the genetic architecture underlying resistance to the biotrophic fungal pathogen <i>Puccinia striiformis</i> f. sp. <i>tritici</i> (wheat stripe rust) in intermediate host and nonhost grass species. In the diploid grass species <i>Hordeum vulgare</i> (barley) and <i>Brachypodium distachyon</i> (purple false brome) we have identified several loci that contribute to limiting the host range of <i>P. striiformis</i>. Mapping of nonhost resistance in both species has found that resistance is conditioned by a few major effect loci. We have fine-mapped six nonhost resistance loci, including <i>Rps6</i>, <i>Rps7</i>, and <i>Rps8</i> in barley, and <i>Yrr1</i>, <i>Yrr2</i>, and <i>Yrr3</i> in <i>B. distachyon</i>. Ongoing work in the group includes map-based cloning of these loci and their association with known and novel components of immunity.

---

## Multiple pathogen recognition of <i>R</i> genes
Collaborations: Tom Wolpert at [Oregon State University](http://bpp.oregonstate.edu/wolpert), Ryohei Terauchi at [Kyoto University](http://www.kais.kyoto-u.ac.jp/english/overview/members/), and Ulrich Schaffrath at [RWTH Aachen University](http://www.bio3.rwth-aachen.de/)


---

## Molecular tradeoffs in immunity to biotrophs and necrotrophs
Collaboration: Tom Wolpert at [Oregon State University](http://bpp.oregonstate.edu/wolpert)

Breeding for disease resistance often involves the introgression of resistance (<i>R</i>) genes into susceptible commercial crop cultivars. During the 1940s, oat crown rust (<i>Puccinia coronata</i>) resistance derived from the oat (<i>Avena sativa</i>) cultivar Victoria was introduced into elite germplasm. Subsequently, Victoria and its derivatives were grown throughout the US Midwest. Unbeknownst to plant breeders, resistance conferred by the <i>R</i> gene <i>Pc2</i> and its introduction facilitated a host jump of the necrotrophic pathogen <i>Cochliobolus victoriae</i> from timothy-grass (<i>Phleum pratense</i>) onto oat. This jump caused a massive epidemic, creating a situation where a novel disease spread across oat fields and caused farms to switch to completely different cultivars in subsequent years. Virulence of <i>C. victoriae</i> on oat requires two genetic factors: (1) the susceptibility gene <i>Victoria blight</i> (<i>Vb</i>), which is in coupling with <i>Pc2</i> and (2) the host-selective toxin victorin, which exists as several complex chlorinated cyclic pentapeptides. Oat is a difficult genetic system, but the discovery that victorin elicits sensitivity in several plant species including barley, <i>Brachypodium distachyon</i>, rice, <i>Phaseolus vulgaris</i>, and critically, <i>Arabidopsis thaliana</i>, meant that these systems could be used to understand how victorin induces susceptibility to necrotrophs. In <i>Arabidopsis thaliana</i>, it was shown that <i>LOV1</i> (<i>LOCUS ORCHESTRATING VICTORIN EFFECTS1</i>) conditioning victorin sensitivity encodes a coiled-coil, nucletotide binding, leucine-rich repeat (CC-NB-LRR) protein. NB-LRR genes are typically involved in disease resistance and this observation highlights a mechanism of exploitating the plant immune response to aid necrotrophic pathogen infection. Ongoing work in the group includes the identification of the genes controlling victorin sensitivity in barley (<i>HvLov1</i>) and <i>B. distachyon</i> (<i>BdLov1</i>). In addition, we are current taking a RenSeq/PacBio-based approach to mine the NB-LRRome of oat and use natural and induced variation to identify <i>Pc2</i>/<i>Vb</i>.

---

## Suppression of resistance in polyploid genomes
Collaborations: Colin Hiebert at [Agriculture and Agri-Food Canada](http://www.agr.gc.ca/eng/science-and-innovation/research-centres/manitoba/morden-research-and-development-centre/scientific-staff-and-expertise/hiebert-colin-phd/?id=1272636174163) and Wolfgang Spielmeyer at [CSIRO, Black Mountain](https://www.csiro.au/en/Locations/ACT/Black-Mountain)

 

---

## Genomics of immunity in cereals

Positional cloning of resistance (<i>R</i>) loci to biotrophic pathogens in angiosperms has revealed that the majority of genes encode proteins with two critical domains: a nucleotide binding (NB) domain and tandem arrays of leucine rich repeats (LRRs). Localized to the cytoplasm and nucleus, NB-LRRs recognize the presence of pathogens through direct recognition of pathogen-derived elicitors or through indirect recognition of host proteins that are modified by the pathogen. NB-LRR encoding loci are highly complex, exhibiting substantial copy number variation, structural rearrangements, and novel gene content between allelic variants. Collectively, these characteristics render this gene family a major challenge with modern assembly and annotation from next-generation sequencing. Using the Morex whole genome shotgun reference assembly as a starting point, we identified 328 NB-LRRs are present in the annotated Morex genome. The NB-LRRs are unequally distributed across the genome, with particular enrichment on chromosome arms: 1HS, 2HS, 6HL, and 7HS. We discovered that the annotated NB-LRRs were incomplete and set out to improve the annotation of this gene family. Using an approach that integrates transcriptome-based discovery pipeline and RenSeq/PacBio (an exome capture that specifically targets the NB-LRRs), we have identified over 600 NB-LRR encoding genes. We are now leveraging the leaf transcriptomes derived from 39 elite, landrace, and wild barley accessions to assess allelic diversity including phylogenetic relationship and degree of presence/absence variation relative to the reference genome.

---

## Resistance gene discovery in barley
Collaborations: Brian Steffenson at [University of Minnesota](https://plpa.cfans.umn.edu/people/faculty/brian-steffenson)


Barley stripe rust (=barley yellow rust), caused by <i>Puccinia striiformis</i> f. sp. <i>hordei</i>, is a destructive fungal pathogen that significantly affects worldwide barley cultivation. A major constraint in breeding resistant cultivars is the lack of mapping information of resistance (<i>R</i>) genes and their introgression into adapted germplasm. A considerable number of <i>R</i> genes have been described in barley to <i>P. striiformis</i> f. sp. <i>hordei</i>, but only a few loci have been mapped. Current work in the group involves the mapping of over 10 different <i>Resistance to P. striiformis</i> (<i>Rps</i>) and the generation of a near-isogenic panel using the recurrent cultivar Manchuria.

---

## Pre-zygotic and post-zygotic hybridization barriers
Collaborations: Sebastian Schornack at [Sainsbury Laboratory, Cambridge University](http://www.slcu.cam.ac.uk/research/schornack-group)

<i>Gaf2</i>
<i>Lep1</i>

---

## Genetic resource development
Collaborations: John Doonan at [Aberystwyth University](https://www.aber.ac.uk/en/ibers/staff-profiles/listing/profile/jhd2) and Patrick Hayes at [Oregon State University](http://barleyworld.org/)

Recombinant inbred line (RIL) and doubled-haploid (DH) mapping populations remain a critical resource for gene discovery. In both barley and <i>Brachypodium distachyon</i>, we are involved in the generation of several mapping populations for the community. In barley we are developing the Oregon Promise DH population (OPP), which is a cross between the 2-row malting (particularly for distilling) and transformable cultivar Golden Promise and the 2-row craft malting cultivar Full Pint. The OPP segregates for a range of morphological, disease resistance, and malting traits. In <i>B. distachyon</i> we are developing the ABR6 x Bd21 RIL population, which represents an extreme cross between the Spanish accession ABR6 and the Iraqi accession Bd21 (also the gold-standard sequenced reference accession). These populations are open to the community, please contact us for access.

Last updated: 2017 March 25
