<!DOCTYPE html>
<html>
    
    <!--
	Ion by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<head>

    <title>Publications</title>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <meta name="description" content="" />
    <meta name="keywords" content="" />
    <!-- Favicons -->
    <link rel="shortcut icon" type="image/png" href="http://localhost:4000/favicon.png" />
    <link rel="shortcut icon" href="http://localhost:4000/favicon.ico" />
    <!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
    <script src="/js/jquery.min.js"></script>
    <script src="/js/skel.min.js"></script>
    <script src="/js/skel-layers.min.js"></script>
    <script src="/js/init.js"></script>
    <noscript>
        <link rel="stylesheet" href="/css/skel.css" />
        <link rel="stylesheet" href="/css/style.css" />
        <link rel="stylesheet" href="/css/style-xlarge.css" />
    	<link rel="stylesheet" href="/css/academicons.css"/>
    </noscript>
</head>


	<body id="top">

        <!-- Header -->
<header id="header" class="skel-layers-fixed">
    <h1><a href="/" class="site-title" >Moscou Group at The Sainsbury Laboratory</a></h1>
    <nav id="nav">
        <ul>
            <li><a href="/">Home</a></li>
	    <li><a href="/Research">Research</a></li>
	    <li><a href="/Publications">Publications</a></li>
	    <li><a href="/Funding">Funding</a></li>
	    <li><a href="/Blog">Blog</a></li>
	    <li><a href="/comics">Comics</a></li>
	    <li><a href="/CV">CV</a></li>
	    <li><a href="/Group/index.html">Group</a></li>
        </ul>
    </nav>
</header>


        <!-- Main -->
<section id="banner">
    <header class="inner">
        <h2>Publications</h2>
    </header>
</section>
<section id="main" class="wrapper style1">
    <div class="container">
	    <h2 id="preprints-at-biorxiv">Preprints at <a href="https://www.biorxiv.org/">bioRxiv</a></h2>
<p>Brabham, H. J., Hernández-Pinzón, I., Holden, S., Lorang, J., and Moscou, M.J. (2017) An ancient integration in a plant NLR is maintained as a <i>trans</i>-species polymorphism. <b><i>bioRxiv</i></b> 239541 <a href="http://doi.org/10.1101/239541">doi:10.1101/239541</a></p>
<ul>
  <li><a href="https://figshare.com/account/home#/projects/27682">Raw data for figures and <i>de novo</i> assembled transcriptomes on figshare</a></li>
  <li><a href="http://itol.embl.de/">Phylogenetic trees on iTOL</a> - Under ‘Sharing data’ use the iTOL login: moscou</li>
  <li><a href="https://github.com/matthewmoscou/Exo70">Scripts and Exo70 annotation</a></li>
  <li><a href="https://www.ncbi.nlm.nih.gov">Raw sequencing and assembled genomic and transcriptomic data</a>
    <ol>
      <li><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA378334/">Barley RNAseq set 1</a></li>
      <li><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA378723/">Barley RNAseq set 2</a></li>
      <li><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA422803/">Oat RNAseq and RenSeq/PacBio</a></li>
      <li><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA422986/">Barley RenSeq/PacBio</a></li>
    </ol>
  </li>
</ul>

<p>Bailey, P.C., Schudoma, C., Jackson, W., Baggs, E., Dagdas, G., Haerty, W., Moscou, M., Krasileva, K.V. (2017) Dominant integration locus drives continuous diversification of plant immune receptors with exogenous domain fusions. <b><i>bioRxiv</i></b> 100834 <a href="https://doi.org/10.1101/100834">doi:10.1101/100834</a></p>
<ul>
  <li><a href="http://itol.embl.de/">Phylogenetic trees on iTOL</a> - Under ‘Sharing data’ use the iTOL login: KrasilevaGroup</li>
  <li><a href="https://github.com/krasileva-group/plant_rgenes">Scripts for NLR-ID identification</a></li>
  <li><a href="https://github.com/krasileva-group/tandem">Scripts for tandem NLR analysis</a></li>
  <li><a href="https://github.com/matthewmoscou/NLR-ID_motif">Scripts for CID domain analysis</a></li>
</ul>

<p>Clare, S., Kitcher, W., Gardiner, M., Green, P., Hubbard, A., and Moscou, M.J. (2016) Defining the genetic architecture of stripe rust resistance in the barley accession HOR 1428. <b><i>bioRxiv</i></b> 093773 <a href="https://doi.org/10.1101/093773">doi:10.1101/093773</a></p>

<h2 id="2018">2018</h2>
<p>Bailey, P.C., Schudoma, C., Jackson, W., Baggs, E., Dagdas, G., Haerty, W., Moscou, M., Krasileva, K.V. 2018. Dominant integration locus drives continuous diversification of plant immune receptors with exogenous domain fusions. <b><i>Genome Biology</i></b> <b>19</b>:23. <a href="https://doi.org/10.1186/s13059-018-1392-6">doi:10.1186/s13059-018-1392-6</a></p>

<h2 id="2017">2017</h2>
<p>Moscou, M. J. and van Esse, H. P. (2017) The quest for durable resistance. <b><i>Science</i></b> <b>358</b>:1541-1542. <a href="http://science.sciencemag.org/cgi/content/full/358/6370/1541?ijkey=/LGSoXCyQFClw&amp;keytype=ref&amp;siteid=sci">doi:10.1126/science.aar4797</a></p>

<p>Herb, D.W., Benson, A., Carey, D., Cistue, L., Filichkin, T, Fisk, S.P., Helgerson, L., Jennings, R., Li, Y., Meints, B.M., Monsour, R., Moscou, M., Nguygen, A., Onio, A., Romagosa, I., Thiel, R., Thomas, W.B., Tynan, S.P., Vega, V., Vinkemeier, K., Hayes, P.M. (2017) Malt modification and its effects on the contributions of barley genotype to beer flavor. <b><i>Journal of American Society of Brewing Chemists</i></b>. <b>4</b>:354-362 <a href="http://doi.org/10.1094/ASBCJ-2017-4976-01">doi:10.1094/ASBCJ-2017-4976-01</a></p>

<p>Herb, D.W., Meints, B.M., Jennings, R., Romagosa, I., Moscou, M., Carey, D., Cistue, L., Filichkin, T, Fisk, S.P., Helgerson, L., Martens, C., Monsour, R., Thiel, R., Tynan, S., Thomas, W.T.B., Vinkemeier, K., Hayes, P.M. 2017. Effects of barley (<i>Hordeum vulgare</i> L.) variety and growing environment on beer flavor. Journal of American Society of Brewing Chemists. <b>4</b>:345-353 <a href="http://doi.org/10.1094/ASBCJ-2017-4860-01">doi:10.1094/ASBCJ-2017-4860-01</a></p>

<p>Yu, G., Champouret, N., Steuernagel, B., Olivera, P.D., Simmons, J. Williams, C. Johnson, R., Moscou, M.J., Hernández-Pinzón, I., Green, P., Sela, H., Millet, E., Jones, J.D.G., Ward, E.R., Steffenson, B.J., and Wulff, B.B.H. (2017) Discovery and characterization of two new stem rust resistance genes in <i>Aegilops sharonensis</i>. <b><i>Theoretical and Applied Genetics</i></b> <b>130</b>:1207–1222 <a href="https://doi.org/10.1007/s00122-017-2882-8">doi:10.1007/s00122-017-2882-8</a></p>

<p>Hisano, H., Meints, B., Moscou, M.J., Cistue, L., Echávarri, B., Sato, K., and Hayes, P.M. (2017) Selection of transformation-efficient barley genotypes based on <i>TFA</i> (transformation amenability) haplotype and higher resolution mapping of the <i>TFA</i> loci <b><i>Plant Cell Reports</i></b> <b>36</b>:611–620 <a href="https://doi.org/10.1007/s00299-017-2107-2">doi:10.1007/s00299-017-2107-2</a></p>

<p>Bettgenhaeuser, J., Corke, F.M.K., Opanowicz, M., Green, P., Hernández-Pinzón, I., Doonan, J.H., Moscou, M.J. (2017) Natural variation in <i>Brachypodium</i> links vernalization and flowering time loci as major flowering determinants. <b><i>Plant Physiology</i></b> <b>173</b>:256-268 <a href="https://doi.org/10.1104/pp.16.00813">doi:10.1104/pp.16.00813</a></p>

<h2 id="2016">2016</h2>
<p>Le Fevre, R., O’Boyle, B., Moscou, M.J., and Schornack S. (2016) Colonization of barley by the broad-host hemibiotrophic pathogen <i>Phytophthora palmivora</i> uncovers a leaf development dependent involvement of <i>MLO</i>. <b><i>Molecular Plant-Microbe Interactions</i></b> <b>29</b>: 385-395 <a href="https://doi.org/10.1094/MPMI-12-15-0276-R">doi:10.1094/MPMI-12-15-0276-R</a></p>

<p>Dawson, A.M., Ferguson, J., Gardiner, M., Green, P., Hubbard, A., and Moscou, M.J. (2016) Isolation and fine mapping of <i>Rps6</i>: An intermediate host resistance gene in barley to wheat stripe rust. <b><i>Theoretical and Applied Genetics</i></b> <b>129</b>: 831-843 <a href="https://doi.org/10.1007/s00122-015-2659-x">doi:10.1007/s00122-015-2659-x</a></p>

<h2 id="2015">2015</h2>
<p>Dawson, A.M., Bettgenhaeuser, J., Gardiner, M., Green, P., Hernández-Pinzón, I., Hubbard, A., and Moscou, M.J. (2015) The development of quick, robust, quantitative phenotypic assays for describing the host-nonhost landscape to stripe rust. <b><i>Frontiers in Plant Science</i></b> <b>6</b>: 876 <a href="https://doi.org/10.3389/fpls.2015.00876">doi:10.3389/fpls.2015.00876</a></p>

<p>Muñoz-Amatriaín, M., Lonardi, S., Luo, M. C., Madishetty, K., Svensson, J.T., Moscou, M.J., Wanamaker, S., Jiang, T., Kleinhofs, A., Muehlbauer, G.J., Wise, R., Stein, N., Ma, Y., Rodriguez, E., Kudrna, D., Bhat, P., Chao, S., Condamine, P., Heinen, S., Resnik, J., Wing, R., Witt, H., Alpert, M., Beccuti, M., Bozdag, S., Cordero, F., Mirebrahim, H., Ounit, R., Wu, Y., You, F., Zheng, J., Doležel, J., Grimwood, J., Schmutz, J., Duma, D., Altschmied, L., Blake, T., Bregitzer, P., Cooper, L., Dilbirligi, M., Falk, A., Feiz, L., Graner, A., Gustafson, P., Hayes, P.M., Lemaux, P., Mammadov, J., and Close, T.J. (2015) Sequencing of 15 622 gene-bearing BACs clarifies the gene-dense regions of the barley genome. <b><i>The Plant Journal</i></b> <b>84</b>: 216-227. <a href="https://doi.org/doi:10.1111/tpj.12959">doi:10.1111/tpj.12959</a></p>

<p>Knight, E., Binnie, A., Draeger, T., Moscou, M., Rey, M., Sucher, J., Mehra, S., King, I., and Moore, G. (2015) Mapping the ‘breaker’ element of the gametocidal locus proximal to a block of sub-telomeric heterochromatin on the long arm of chromosome 4S<sup>sh</sup> of <i>Aegilops sharonensis</i>. <b><i>Theoretical and Applied Genetics</i></b> <b>128</b>(6):1049-1059 <a href="https://doi.org/10.1007/s00122-015-2489-x">doi:10.1007/s00122-015-2489-x</a></p>

<h2 id="2014">2014</h2>
<p>Wulff, B.H.H. and Moscou, M.J. (2014) Strategies for transferring resistance into wheat: from wide crosses to GM cassettes. <b><i>Frontiers in Plant Science</i></b> <b>5</b>:692 <a href="https://doi.org/10.3389/fpls.2014.00664">doi:10.3389/fpls.2014.00664</a></p>

<p>Bettgenhaeuser, J., Gilbert, M., Ayliffe, M., Moscou, M.J. (2014) Nonhost resistance to rust pathogens – a continuation of continua. <b><i>Frontiers in Plant Science</i></b> <b>5</b>:664 <a href="https://doi.org/10.3389/fpls.2014.00664">doi:10.3389/fpls.2014.00664</a></p>

<h2 id="2013">2013</h2>
<p>Bouyioukos, C., Moscou, M.J., Champouret, N., Hernández-Pinzón I., Ward E.R. and Wulff B.B.H. (2013) Characterisation and analysis of the <i>Aegilops sharonensis</i> transcriptome, a wild relative of wheat in the Sitopsis section. <b><i>PLoS ONE</i><b> <b>8</b>(8): e72782. <a href="https://doi.org/10.1371/journal.pone.0072782">doi:10.1371/journal.pone.0072782</a></b></b></p>

<p>Ayliffe, M. Singh, D., Park, R., Moscou, M., Pryor, T. (2013) The infection of <i>Brachypodium distachyon</i> with selected grass rust pathogens. <b><i>Molecular Plant-Microbe Interactions</i></b> <b>26</b>(8): 946-957 <a href="https://doi.org/10.1094/MPMI-01-13-0017-R">doi:10.1094/MPMI-01-13-0017-R</a></p>

<p>Schornack, S., Moscou, M.J., Ward, E.R., and Horvath, D. M. (2013) Engineering plant disease resistance based on TAL effectors. <b><i>Annual Review of Phytopathology</i></b> <b>51</b>:383-406 <a href="https://doi.org/10.1146/annurev-phyto-082712-102255">doi:10.1146/annurev-phyto-082712-102255</a></p>

<h2 id="2012">2012</h2>
<p>The International Barley Genome Sequencing Consortium (IBSC). (2012) A physical, genetic and functional sequence assembly of the barley genome. <b><i>Nature</i></b> <b>491</b>: 711-716 <a href="https://doi.org/10.1038/nature11543">doi:10.1038/nature11543</a></p>

<p>Moeller, J.R., Moscou, M.J., Bancroft, T., Skadsen, R.W., Wise, R.P., and Whitham, S.A. (2012) Differential accumulation of host mRNAs on polyribosomes during obligate pathogen-plant interactions. <b><i>Molecular BioSystems</i></b> <b>8</b>:2153-2165 <a href="https://doi.org/10.1039/C2MB25014D">doi:10.1039/C2MB25014D</a></p>

<h2 id="2011">2011</h2>
<p>Muñoz-Amatriaín, M.<sup><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/yin_yang.png" alt="alt text" title="Authors contributed equally" height="12px" width="12px" /></sup>, Moscou, M.J.<sup><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/yin_yang.png" alt="alt text" title="Authors contributed equally" height="12px" width="12px" /></sup>, Bhat, P.R., Svensson, J.T., Bartoš, J., Suchánková, P., Šimková, H., Endo, T.R., Fenton, R. D., Wu, Y., Lonardi, S., Castillo, A.M., Chao, S., Cistué, L., Cuesta-Marcos, A., Forrest, K., Hayden, M.J., Hayes, P.M., Horsley, R.D., Kleinhofs, A., Moody, D., Sato, K., Vallés, M.P., Wulff, B.B.H., Muehlbauer, G.J., Doležel, J., and Close, T.J. (2011) An improved consensus linkage map of barley based on flow-sorted chromosomes and SNP markers. <b><i>The Plant Genome</i></b> <b>4</b>(3): 238-249 <a href="https://doi.org/10.3835/plantgenome2011.08.0023">doi:10.3835/plantgenome2011.08.0023</a></p>

<p>Moscou, M.J., Lauter, N., Steffenson, B., and Wise, R.P. (2011) Quantitative and qualitative stem rust resistance factors in barley are associated with transcriptional suppression of defense regulons. <b><i>PLoS Genetics</i></b> <b>7</b>(7): e1002208 <a href="https://doi.org/10.1371/journal.pgen.1002208">doi:10.1371/journal.pgen.1002208</a></p>

<p>Moscou, M.J., Lauter, N., Caldo, R.A., Nettleton, D., and Wise, R.P. (2011) Quantitative and temporal definition of the <i>Mla</i> transcriptional regulon during barley-powdery mildew interactions.  <b><i>Molecular Plant-Microbe Interactions</i></b> <b>24</b>(6): 694-705 <a href="https://doi.org/10.1094/MPMI-09-10-0211">doi:10.1094/MPMI-09-10-0211</a></p>

<h2 id="2009">2009</h2>
<p>Close, T.J., Bhat, P.R., Lonardi, S., Wu, Y., Rostoks, N., Ramsay, L., Druka, A., Stein, N., Svensson, J.T., Wanamaker, S., Bozdag, S., Roose, M.L., Moscou, M.J., Chao, S., Varshney, R., Szűcs, P., Sato, K., Hayes, P.M., Matthews, D.E., Kleinhofs, A., Muehlbauer, G. J., DeYoung, J., Marshall, D.F., Madishetty, K., Fenton, R.D., Condamine, P., Graner, A., Waugh, R. 2009. Development and implementation of high-throughput SNP genotyping in barley. <b><i>BMC Genomics</i></b> <b>10</b>: 582 <a href="https://doi.org/10.1186/1471-2164-10-582">doi:10.1186/1471-2164-10-582</a></p>

<p>Moscou, M.J. and Bogdanove, A. (2009) A simple cipher governs DNA recognition by TAL effectors. <b><i>Science</i></b> <b>326</b>: 1501 <a href="https://doi.org/10.1126/science.1178817">doi:10.1126/science.1178817</a></p>

<p>Xi, L., Moscou, M.J., Meng, Y., Xu, W. Caldo, R.A., Shaver, M., Nettleton, D., Wise, R.P. (2009) Transcript-based cloning of <i>HvRRP46</i>, a regulator of rRNA processing and <i>R</i> gene-independent cell death in barley-powdery mildew interactions. <b><i>Plant Cell</i></b> <b>21</b>: 3280-3295 <a href="https://doi.org/10.1105/tpc.109.066167">doi:10.1105/tpc.109.066167</a></p>

<p>Meng, Y.<sup><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/yin_yang.png" alt="alt text" title="Authors contributed equally" height="12px" width="12px" /></sup>, Moscou, M.J.<sup><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/yin_yang.png" alt="alt text" title="Authors contributed equally" height="12px" width="12px" /></sup>, Wise, R.P. (2009) <i>Blufensin1</i> negatively impacts basal defense in response to barley powdery mildew. <b><i>Plant Physiology</i></b> <b>149</b>(1): 271-285 <a href="https://doi.org/10.1104/pp.108.129031">doi:10.1104/pp.108.129031</a></p>

<h2 id="2008">2008</h2>
<p>Lauter, N., Moscou, M.J., Habinger, J., Moose, S. (2008) Quantitative genetic dissection of shoot architecture traits in maize: Towards a functional genomics approach. <b><i>The Plant Genome</i></b> <b>1</b>(2): 99-110 <a href="https://doi.org/10.3835/plantgenome2008.06.0385">doi:10.3835/plantgenome2008.06.0385</a></p>

<p>Potokina, E., Druka, A., Luo, Z., Moscou, M., Wise, R., Waugh, R., Kearsey, M. (2008) Tissue-dependent limited pleiotropy affects gene expression in barley. <b><i>Plant Journal</i></b> <b>56</b>(2): 287-296 <a href="https://doi.org/10.1111/j.1365-313X.2008.03601.x">doi:10.1111/j.1365-313X.2008.03601.x</a></p>

<h2 id="2007">2007</h2>
<p>Wise, R.P., Moscou, M.J., Bogdanove, A.J., Whitham, S.A. (2007) Transcript profiling in host–pathogen interactions. <b><i>Annual Review of Phytopathology</i></b> <b>45</b>: 329-369 <a href="https://doi.org/10.1146/annurev.phyto.45.011107.143944">doi:10.1146/annurev.phyto.45.011107.143944</a></p>

<p>Last updated: 2018 January 3</p>

    </div>
</section>


	<!-- Footer -->
<footer id="footer">
    <div class="container">
        <div class="row double">
            <div class="6u">
                <div class="row collapse-at-2">
                    <div class="6u">
			<section class="special">
				<a href="http://www.tsl.ac.uk" class="image fit"><img src="http://localhost:4000/images/TSL_RGB.png" alt="" /></a>
			</section>
                    </div>
                    <div class="6u">
			    <h3><a href="http://www.tsl.ac.uk/">The Sainsbury Laboratory</a></h3>
                        <ul class="alt">
                            <li><a href="http://www.tsl.ac.uk/groups/moscou-group/">Moscou Group at TSL</a></li>
                            <li><a href="http://www.tsl.ac.uk/about-tsl/">About TSL</a></li>
                        </ul>
                    </div>
                </div>
            </div>
            <div class="6u">
                <h2>Moscou Group at The Sainsbury Laboratory</h2>
                <p>The Moscou group focuses on understanding immunity in the grasses. Current themes in our research include dissecting the genetic architecture of nonhost resistance, tradeoffs between biotrophs and necrotrophs, seedling versus adult plant resistance, and suppression of resistance in polyploid genomes. The group uses genetics and bioinformatics to explore biology and solve long-standing puzzles in biology.</p>
		<p>The views on this website are those of its author.</p>
                <ul class="icons">
                    <li><a href="https://twitter.com/matthewmoscou?lang=en" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
                    <li><a href="https://github.com/matthewmoscou" class="icon fa-github"><span class="label">Github</span></a></li>
                    <li><a href="https://www.linkedin.com/in/matthew-moscou-543b9b6/" class="icon fa-linkedin"><span class="label">Linkedin</span></a></li>
                    <li><a href="https://www.researchgate.net/profile/Matthew_Moscou" class="ai ai-researchgate"><span class="label">ResearchGate</span></a></li>
                    <li><a href="https://publons.com/a/602996/" class="ai ai-publons"><span class="label">Publons</span></a></li>
                    <li><a href="https://scholar.google.co.uk/citations?user=7Y5ViZ8AAAAJ&hl=en" class="ai ai-google-scholar"><span class="label">Google Scholar</span></a></li>
                    <li><a href="http://biorxiv.org/search/Moscou" class="ai ai-biorxiv"><span class="label">bioRxiv</span></a></li>
                </ul>
            </div>
        </div>
        <ul class="copyright">
            <li>&copy; Matthew Moscou. All rights reserved.</li>
	    <li>Layout <a href="https://github.com/old-jekyll-templates/Ion-Jekyll-Theme">Ion Theme</a></li>
        </ul>
    </div>
</footer>


    </body>

</html>
