<!DOCTYPE html>
<html>
    
    <!--
	Ion by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<head>

    <title>Training</title>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <meta name="description" content="" />
    <meta name="keywords" content="" />
    <!-- Favicons -->
    <link rel="shortcut icon" type="image/png" href="http://localhost:4000/favicon.png" />
    <link rel="shortcut icon" href="http://localhost:4000/favicon.ico" />
    <!--[if lte IE 8]><script src="js/html5shiv.js"></script><![endif]-->
    <script src="/js/jquery.min.js"></script>
    <script src="/js/skel.min.js"></script>
    <script src="/js/skel-layers.min.js"></script>
    <script src="/js/init.js"></script>
    <noscript>
        <link rel="stylesheet" href="/css/skel.css" />
        <link rel="stylesheet" href="/css/style.css" />
        <link rel="stylesheet" href="/css/style-xlarge.css" />
    	<link rel="stylesheet" href="/css/academicons.css"/>
    </noscript>
</head>


	<body id="top">

        <!-- Header -->
<header id="header" class="skel-layers-fixed">
    <h1><a href="/" class="site-title" >Moscou Group at The Sainsbury Laboratory</a></h1>
    <nav id="nav">
        <ul>
            <li><a href="/">Home</a></li>
	    <li><a href="/Research">Research</a></li>
	    <li><a href="/Publications">Publications</a></li>
	    <li><a href="/Funding">Funding</a></li>
	    <li><a href="/Blog">Blog</a></li>
	    <li><a href="/comics">Comics</a></li>
	    <li><a href="/CV">CV</a></li>
	    <li><a href="/Group/index.html">Group</a></li>
        </ul>
    </nav>
</header>


        <!-- Main -->
<section id="banner">
    <header class="inner">
        <h2>Training</h2>
    </header>
</section>
<section id="main" class="wrapper style1">
    <div class="container">
	    <p>Seminal papers that have shaped my scientific experience.
Resources I have accessed over the years that have been extremely useful.</p>

<h2 id="genetics">Genetics</h2>
<h3 id="constructing-genetic-map">Constructing genetic map</h3>

<p><a href="https://www.kyazma.nl/">JoinMap</a></p>

<p>Critical and example papers include:</p>
<blockquote>

</blockquote>

<h3 id="quantitiative-trait-locus-analysis">Quantitiative trait locus analysis</h3>

<p>Critical papers include:</p>
<blockquote>
  <p>Churchill and Doerge (1996) Permutations tests for multiple loci affecting a quantitative character. <strong><em>Genetics</em></strong> <strong>142</strong>:285-294.</p>
</blockquote>

<h3 id="transgressive-segregation">Transgressive segregation</h3>

<p>Critical papers include:</p>
<blockquote>
  <p>deVicente and Tanksley (1993) QTL analysis of transgressive segregation in an interspecific tomato cross. <strong><em>Genetics</em></strong> <strong>134</strong>:585-596.</p>
</blockquote>

<h2 id="genomics">Genomics</h2>

<h3 id="visualization-of-genes-and-genomes">Visualization of genes and genomes</h3>
<p><a href="https://github.com/wilkox/gggenes">gggenes</a><br />
<a href="https://github.com/genotoul-bioinfo/dgenies">dgenies</a> provides dotplots for large genomes<br />
<a href="https://github.com/KamilSJaron/smudgeplot">smudgeplot</a> provides visualization of ploidy and heterozygosity structure</p>

<h2 id="molecular-evolution">Molecular Evolution</h2>
<p><a href="http://evomics.org/learning/phylogenetics/paml/">PAML</a><br />
<a href="http://nbisweden.github.io/MrBayes/">MrBayes</a> uses Bayesian inference and model choice over several phylogenetic and evolutionary models.</p>

<h2 id="plant-microbe-interaction">Plant-Microbe Interaction</h2>

<h3 id="r-genes-versus-qtls"><em>R</em> genes versus QTLs</h3>
<p>There has been a long-standing debate about the potential relationship between <em>R</em> gene-mediated resistance and minor effect QTLs identified based on field-based phenotyping.</p>

<p>Critical papers include:</p>
<blockquote>
  <p>Li <em>et al.</em> (1999) A “defeated” rice resistance gene acts as a QTL against a virulent strain of <em>Xanthomonas oryzae</em> pv. <em>oryzae</em>. <strong><em>Molecular Genomics and Genetics</em></strong> <strong>261</strong>:58-63.</p>
</blockquote>

<h2 id="data-visualization-and-figures">Data, visualization, and figures</h2>
<p><a href="http://dataviz.tools/">Compendium of data visualization tools</a>.<br />
<a href="https://www.vecteezy.com">Vecteezy</a> is a free vector art searchable repository. <br />
<a href="https://github.com/EmilHvitfeldt/r-color-palettes/">Color and color blindness</a> resources.<br />
<a href="https://mjsull.github.io/Easyfig/">Easyfig</a><br />
<a href="https://wilkox.org/gggenes/">gggenes</a><br />
<a href="https://www.r-graph-gallery.com/index.html">Diversity of tools in ggplot</a></p>

<h2 id="high-throughput-sequencing">High-throughput sequencing</h2>
<p><a href="https://www.ebi.ac.uk/~nf/hts_mappers/">Tools for mapping high-throughput sequencing data</a><br />
<a href="https://rnaseq.uoregon.edu/index.html">RNAseqlopedia</a><br />
<a href="https://github.com/griffithlab/rnaseq_tutorial">RNAseq tutorial from the Griffith Lab</a><br />
<a href="https://haroldpimentel.wordpress.com/2014/05/08/what-the-fpkm-a-review-rna-seq-expression-units/">Counts, TPM, RPKM, and FPKM</a><br />
<a href="https://bioinformatics.uconn.edu/genome-size-estimation-tutorial/">Estimating genome size using k-mer frequency</a></p>

<h2 id="bioinformatics">Bioinformatics</h2>
<p><a href="https://datacarpentry.org/">Data Carpentry</a><br />
<a href="https://github.com/bioinformatics-core-shared-training">Bioinformatics Core Training</a><br />
<a href="https://github.com/jduckles/dsskills">Data Science Skills in Research</a></p>

<h2 id="plant-biology">Plant biology</h2>
<p><a href="https://sburgess895713696.wordpress.com/2019/01/19/what-is-plant-biology/">What plants are important</a></p>

<h2 id="statistics">Statistics</h2>
<p><a href="https://github.com/ACCLAB/DABEST-python">DABEST: Data Analysis using Bootstrap-Coupled ESTimation</a><br />
<a href="https://ourcodingclub.github.io/2017/03/15/mixed-models.html">A good tutorial in mixed linear models</a></p>

<h2 id="graphic-design">Graphic design</h2>
<p><a href="www.canvas.com">Canvas</a> design reports, flyers, posters, and other diverse media with a range of templates</p>

<h2 id="e-mail">E-mail</h2>
<p><a href="https://mailchimp.com/">Mailchimp</a> manages large e-mail mailing lists</p>

<p>Last updated: 2019 December 25</p>

    </div>
</section>


	<!-- Footer -->
<footer id="footer">
    <div class="container">
        <div class="row double">
            <div class="6u">
                <div class="row collapse-at-2">
                    <div class="6u">
			<section class="special">
				<a href="http://www.tsl.ac.uk" class="image fit"><img src="http://localhost:4000/images/TSL_RGB.png" alt="" /></a>
			</section>
                    </div>
                    <div class="6u">
			    <h3><a href="http://www.tsl.ac.uk/">The Sainsbury Laboratory</a></h3>
                        <ul class="alt">
                            <li><a href="http://www.tsl.ac.uk/groups/moscou-group/">Moscou Group at TSL</a></li>
                            <li><a href="http://www.tsl.ac.uk/about-tsl/">About TSL</a></li>
                        </ul>
                    </div>
                </div>
            </div>
            <div class="6u">
                <h2>Moscou Group at The Sainsbury Laboratory</h2>
                <p>The Moscou group focuses on understanding immunity in the grasses. Current themes in our research include dissecting the genetic architecture of nonhost resistance, tradeoffs between biotrophs and necrotrophs, seedling versus adult plant resistance, and suppression of resistance in polyploid genomes. The group uses genetics and bioinformatics to explore biology and solve long-standing puzzles in biology.</p>
		<p>The views on this website are those of its author.</p>
                <ul class="icons">
                    <li><a href="https://twitter.com/matthewmoscou?lang=en" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
                    <li><a href="https://github.com/matthewmoscou" class="icon fa-github"><span class="label">Github</span></a></li>
                    <li><a href="https://www.linkedin.com/in/matthew-moscou-543b9b6/" class="icon fa-linkedin"><span class="label">Linkedin</span></a></li>
                    <li><a href="https://www.researchgate.net/profile/Matthew_Moscou" class="ai ai-researchgate"><span class="label">ResearchGate</span></a></li>
                    <li><a href="https://publons.com/a/602996/" class="ai ai-publons"><span class="label">Publons</span></a></li>
                    <li><a href="https://scholar.google.co.uk/citations?user=7Y5ViZ8AAAAJ&hl=en" class="ai ai-google-scholar"><span class="label">Google Scholar</span></a></li>
                    <li><a href="http://biorxiv.org/search/Moscou" class="ai ai-biorxiv"><span class="label">bioRxiv</span></a></li>
                </ul>
            </div>
        </div>
        <ul class="copyright">
            <li>&copy; Matthew Moscou. All rights reserved.</li>
	    <li>Layout <a href="https://github.com/old-jekyll-templates/Ion-Jekyll-Theme">Ion Theme</a></li>
        </ul>
    </div>
</footer>


    </body>

</html>
