<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>CV</title>
  <meta name="description" content="The Moscou group focuses on understanding immunity in the grasses. Current themes in our research include dissecting the genetic architecture of nonhost resi...">

  <link rel="stylesheet" href="/assets/main.css">
  <link rel="canonical" href="http://localhost:4000/CV/">
  <link rel="alternate" type="application/rss+xml" title="Moscou Group at The Sainsbury Laboratory" href="/feed.xml">
  
  
</head>


  <body>

    <header class="site-header" role="banner">

  <div class="wrapper">

    <a class="site-title" href="/">Moscou Group at The Sainsbury Laboratory</a>

    <nav class="site-nav">
      <span class="menu-icon">
        <svg viewBox="0 0 18 15" width="18px" height="15px">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </span>

      <div class="trigger">
        
          
          <a class="page-link" href="/CV/">CV</a>
          
        
          
          <a class="page-link" href="/about/">About</a>
          
        
          
        
          
        
          
        
      </div>
    </nav>

  </div>

</header>


    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <header class="post-header">
    <h1 class="post-title">CV</h1>
  </header>

  <div class="post-content">
    <h1 id="matthew-james-moscou"><b>Matthew James Moscou</b></h1>

<p><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/orcid-logo.png" alt="alt text" title="ORCID ID" height="24px" width="24px" /> <a href="http://orcid.org/0000-0003-2098-6818">http://orcid.org/0000-0003-2098-6818</a></p>

<h2 id="address">Address</h2>
<p>The Sainsbury Laboratory<br />
Norwich Research Park<br />
Norwich<br />
NR4 7UH<br />
United Kingdom</p>

<h2 id="contact-information">Contact information</h2>
<p>Office: +44 (0) 1603 450 296<br />
Email: matthew.moscou@sainsbury-laboratory.ac.uk</p>

<h2 id="education">Education</h2>
<p>Iowa State University, Bioinformatics and Computational Biology, Ph.D., 2010 <br />
University of California, Riverside, Pure Mathematics, B.S., 2004 <br />
University of California, Riverside, Physics, B.S., 2004</p>

<h2 id="dissertation">Dissertation</h2>
<p>Deciphering the transcriptional regulation and response of barley to obligate fungal biotroph invasion<br />
Advisors: Roger Wise and Julie Dickerson</p>

<h2 id="professional-experience">Professional Experience</h2>
<p>2014-present		Group Leader, The Sainsbury Laboratory, UK<br />
2013-2014		Team Leader, 2Blades Group, The Sainsbury Laboratory, UK<br />
2010-2012	Post-doctoral Research Associate, 2Blades Group, The Sainsbury Laboratory, UK<br />
2005-2010		Graduate Research Assistant, Iowa State University<br />
Summer 2009	Summer Intern, Syngenta Seeds, Inc. – Statistical Genetics Research Group<br />
2004-2005		Programmer/Analyst I, University of California, Riverside</p>

<h2 id="teaching-mentoring-and-supervision">Teaching, Mentoring, and Supervision</h2>
<p>Supervisor of four Ph.D. and four Msc students (TSL, 2011-present)<br />
Lecturer at the University of East Anglia for BIO-03C46 (Plant Biotechnology for Sustainable Food Production) (UEA, 2015)<br />
EMBO Practical Course on Plant-Microbe Interactions, Lecturer (TSL, 2012)<br />
Molecular Biology of Plant-Pathogen Interaction, Guest Lecturer (ISU, 2008)<br />
Computational and Systems Biology Summer Institute, Lecturer/Teaching Assistant (ISU, 2007)<br />
National Science Foundation, Research Experience for Teachers, Mentor (ISU, 2006)<br />
Bioinformatics and Computational Biology Summer Institute, Lecturer/Teaching Assistant (ISU, 2006)</p>

<h2 id="fellowships-and-awards">Fellowships and Awards</h2>
<p>Human Frontier Science Program Fellow (www.hfsp.org), Long-Term Fellowship, 2011-2014<br />
Graduate Research Excellence Award, Iowa State University, 2010<br />
BioVision.Nxt Fellow (http://www.biovision.org), 2009<br />
James Cornette Research Fellowship in Bioinformatics and Computational Biology, 2008</p>

<h2 id="publications">Publications</h2>
<h3 id="2017">2017</h3>
<p>Yu, G., Champouret, N., Steuernagel, B., Olivera, P.D., Simmons, J. Williams, C. Johnson, R., Moscou, M.J., Hernández-Pinzón, I., Green, P., Sela, H., Millet, E., Jones, J.D.G., Ward, E.R., Steffenson, B.J., and Wulff, B.B.H. (2017) Discovery and characterization of two new stem rust resistance genes in <i>Aegilops sharonensis</i>. <b><i>Theoretical and Applied Genetics</i></b> Accepted. <a href="https://doi.org/10.1007/s00122-017-2882-8">doi:10.1007/s00122-017-2882-8</a></p>

<p>Hisano, H., Meints, B., Moscou, M.J., Cistue, L., Echávarri, B., Sato, K., and Hayes, P.M. (2017) Selection of transformation-efficient barley genotypes based on <i>TFA</i> (transformation amenability) haplotype and higher resolution mapping of the <i>TFA</i> loci <b><i>Plant Cell Reports</i></b> Accepted. <a href="https://doi.org/10.1007/s00299-017-2107-2">doi:10.1007/s00299-017-2107-2</a></p>

<p>Bettgenhaeuser, J., Corke, F.M.K., Opanowicz, M., Green, P., Hernández-Pinzón, I., Doonan, J.H., Moscou, M.J. (2017) Natural variation in <i>Brachypodium</i> links vernalization and flowering time loci as major flowering determinants. <b><i>Plant Physiology</i></b> <b>173</b>:256-268 <a href="https://doi.org/10.1104/pp.16.00813">doi:10.1104/pp.16.00813</a></p>

<h3 id="2016">2016</h3>
<p>Le Fevre, R., O’Boyle, B., Moscou, M.J., and Schornack S. (2016) Colonization of barley by the broad-host hemibiotrophic pathogen <i>Phytophthora palmivora</i> uncovers a leaf development dependent involvement of <i>MLO</i>. <b><i>Molecular Plant-Microbe Interactions</i></b> <b>29</b>: 385-395 <a href="https://doi.org/10.1094/MPMI-12-15-0276-R">doi:10.1094/MPMI-12-15-0276-R</a></p>

<p>Dawson, A.M., Ferguson, J., Gardiner, M., Green, P., Hubbard, A., and Moscou, M.J. (2016) Isolation and fine mapping of <i>Rps6</i>: An intermediate host resistance gene in barley to wheat stripe rust. <b><i>Theoretical and Applied Genetics</i></b> <b>129</b>: 831-843 <a href="https://doi.org/10.1007/s00122-015-2659-x">doi:10.1007/s00122-015-2659-x</a></p>

<h3 id="2015">2015</h3>
<p>Dawson, A.M., Bettgenhaeuser, J., Gardiner, M., Green, P., Hernández-Pinzón, I., Hubbard, A., and Moscou, M.J. (2015) The development of quick, robust, quantitative phenotypic assays for describing the host-nonhost landscape to stripe rust. <b><i>Frontiers in Plant Science</i></b> <b>6</b>: 876 <a href="https://doi.org/10.3389/fpls.2015.00876">doi:10.3389/fpls.2015.00876</a></p>

<p>Muñoz-Amatriaín, M., Lonardi, S., Luo, M. C., Madishetty, K., Svensson, J.T., Moscou, M.J., Wanamaker, S., Jiang, T., Kleinhofs, A., Muehlbauer, G.J., Wise, R., Stein, N., Ma, Y., Rodriguez, E., Kudrna, D., Bhat, P., Chao, S., Condamine, P., Heinen, S., Resnik, J., Wing, R., Witt, H., Alpert, M., Beccuti, M., Bozdag, S., Cordero, F., Mirebrahim, H., Ounit, R., Wu, Y., You, F., Zheng, J., Doležel, J., Grimwood, J., Schmutz, J., Duma, D., Altschmied, L., Blake, T., Bregitzer, P., Cooper, L., Dilbirligi, M., Falk, A., Feiz, L., Graner, A., Gustafson, P., Hayes, P.M., Lemaux, P., Mammadov, J., and Close, T.J. (2015) Sequencing of 15 622 gene-bearing BACs clarifies the gene-dense regions of the barley genome. <b><i>The Plant Journal</i></b> <b>84</b>: 216-227. <a href="https://doi.org/doi:10.1111/tpj.12959">doi:10.1111/tpj.12959</a></p>

<p>Knight, E., Binnie, A., Draeger, T., Moscou, M., Rey, M., Sucher, J., Mehra, S., King, I., and Moore, G. (2015) Mapping the ‘breaker’ element of the gametocidal locus proximal to a block of sub-telomeric heterochromatin on the long arm of chromosome 4S<sup>sh</sup> of <i>Aegilops sharonensis</i>. <b><i>Theoretical and Applied Genetics</i></b> <b>128</b>(6):1049-1059 <a href="https://doi.org/10.1007/s00122-015-2489-x">doi:10.1007/s00122-015-2489-x</a></p>

<h3 id="2014">2014</h3>
<p>Wulff, B.H.H. and Moscou, M.J. (2014) Strategies for transferring resistance into wheat: from wide crosses to GM cassettes. <b><i>Frontiers in Plant Science</i></b> <b>5</b>:692 <a href="https://doi.org/10.3389/fpls.2014.00664">doi:10.3389/fpls.2014.00664</a></p>

<p>Bettgenhaeuser, J., Gilbert, M., Ayliffe, M., Moscou, M.J. (2014) Nonhost resistance to rust pathogens – a continuation of continua. <b><i>Frontiers in Plant Science</i></b> <b>5</b>:664 <a href="https://doi.org/10.3389/fpls.2014.00664">doi:10.3389/fpls.2014.00664</a></p>

<h3 id="2013">2013</h3>
<p>Bouyioukos, C., Moscou, M.J., Champouret, N., Hernández-Pinzón I., Ward E.R. and Wulff B.B.H. (2013) Characterisation and analysis of the <i>Aegilops sharonensis</i> transcriptome, a wild relative of wheat in the Sitopsis section. <b><i>PLoS ONE</i><b> <b>8</b>(8): e72782. <a href="https://doi.org/10.1371/journal.pone.0072782">doi:10.1371/journal.pone.0072782</a></b></b></p>

<p>Ayliffe, M. Singh, D., Park, R., Moscou, M., Pryor, T. (2013) The infection of <i>Brachypodium distachyon</i> with selected grass rust pathogens. <b><i>Molecular Plant-Microbe Interactions</i></b> <b>26</b>(8): 946-957 <a href="https://doi.org/10.1094/MPMI-01-13-0017-R">doi:10.1094/MPMI-01-13-0017-R</a></p>

<p>Schornack, S., Moscou, M.J., Ward, E.R., and Horvath, D. M. (2013) Engineering plant disease resistance based on TAL effectors. <b><i>Annual Review of Phytopathology</i></b> <b>51</b>:383-406 <a href="https://doi.org/10.1146/annurev-phyto-082712-102255">doi:10.1146/annurev-phyto-082712-102255</a></p>

<h3 id="2012">2012</h3>
<p>The International Barley Genome Sequencing Consortium (IBSC). (2012) A physical, genetic and functional sequence assembly of the barley genome. <b><i>Nature</i></b> <b>491</b>: 711-716 <a href="https://doi.org/10.1038/nature11543">doi:10.1038/nature11543</a></p>

<p>Moeller, J.R., Moscou, M.J., Bancroft, T., Skadsen, R.W., Wise, R.P., and Whitham, S.A. (2012) Differential accumulation of host mRNAs on polyribosomes during obligate pathogen-plant interactions. <b><i>Molecular BioSystems</i></b> <b>8</b>:2153-2165 <a href="https://doi.org/10.1039/C2MB25014D">doi:10.1039/C2MB25014D</a></p>

<h3 id="2011">2011</h3>
<p>Muñoz-Amatriaín, M.<sup><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/yin_yang.png" alt="alt text" title="Authors contributed equally" height="12px" width="12px" /></sup>, Moscou, M.J.<sup><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/yin_yang.png" alt="alt text" title="Authors contributed equally" height="12px" width="12px" /></sup>, Bhat, P.R., Svensson, J.T., Bartoš, J., Suchánková, P., Šimková, H., Endo, T.R., Fenton, R. D., Wu, Y., Lonardi, S., Castillo, A.M., Chao, S., Cistué, L., Cuesta-Marcos, A., Forrest, K., Hayden, M.J., Hayes, P.M., Horsley, R.D., Kleinhofs, A., Moody, D., Sato, K., Vallés, M.P., Wulff, B.B.H., Muehlbauer, G.J., Doležel, J., and Close, T.J. (2011) An improved consensus linkage map of barley based on flow-sorted chromosomes and SNP markers. <b><i>The Plant Genome</i></b> <b>4</b>(3): 238-249 <a href="https://doi.org/10.3835/plantgenome2011.08.0023">doi:10.3835/plantgenome2011.08.0023</a></p>

<p>Moscou, M.J., Lauter, N., Steffenson, B., and Wise, R.P. (2011) Quantitative and qualitative stem rust resistance factors in barley are associated with transcriptional suppression of defense regulons. <b><i>PLoS Genetics</i></b> <b>7</b>(7): e1002208 <a href="https://doi.org/10.1371/journal.pgen.1002208">doi:10.1371/journal.pgen.1002208</a></p>

<p>Moscou, M.J., Lauter, N., Caldo, R.A., Nettleton, D., and Wise, R.P. (2011) Quantitative and temporal definition of the <i>Mla</i> transcriptional regulon during barley-powdery mildew interactions.  <b><i>Molecular Plant-Microbe Interactions</i></b> <b>24</b>(6): 694-705 <a href="https://doi.org/10.1094/MPMI-09-10-0211">doi:10.1094/MPMI-09-10-0211</a></p>

<h3 id="2009">2009</h3>
<p>Close, T.J., Bhat, P.R., Lonardi, S., Wu, Y., Rostoks, N., Ramsay, L., Druka, A., Stein, N., Svensson, J.T., Wanamaker, S., Bozdag, S., Roose, M.L., Moscou, M.J., Chao, S., Varshney, R., Szűcs, P., Sato, K., Hayes, P.M., Matthews, D.E., Kleinhofs, A., Muehlbauer, G. J., DeYoung, J., Marshall, D.F., Madishetty, K., Fenton, R.D., Condamine, P., Graner, A., Waugh, R. 2009. Development and implementation of high-throughput SNP genotyping in barley. <b><i>BMC Genomics</i></b> <b>10</b>: 582 <a href="https://doi.org/10.1186/1471-2164-10-582">doi:10.1186/1471-2164-10-582</a></p>

<p>Moscou, M.J. and Bogdanove, A. (2009) A simple cipher governs DNA recognition by TAL effectors. <b><i>Science</i></b> <b>326</b>: 1501 <a href="https://doi.org/10.1126/science.1178817">doi:10.1126/science.1178817</a></p>

<p>Xi, L., Moscou, M.J., Meng, Y., Xu, W. Caldo, R.A., Shaver, M., Nettleton, D., Wise, R.P. (2009) Transcript-based cloning of <i>HvRRP46</i>, a regulator of rRNA processing and <i>R</i> gene-independent cell death in barley-powdery mildew interactions. <b><i>Plant Cell</i></b> <b>21</b>: 3280-3295 <a href="https://doi.org/10.1105/tpc.109.066167">doi:10.1105/tpc.109.066167</a></p>

<p>Meng, Y.<sup><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/yin_yang.png" alt="alt text" title="Authors contributed equally" height="12px" width="12px" /></sup>, Moscou, M.J.<sup><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/yin_yang.png" alt="alt text" title="Authors contributed equally" height="12px" width="12px" /></sup>, Wise, R.P. (2009) <i>Blufensin1</i> negatively impacts basal defense in response to barley powdery mildew. <b><i>Plant Physiology</i></b> <b>149</b>(1): 271-285 <a href="https://doi.org/10.1104/pp.108.129031">doi:10.1104/pp.108.129031</a></p>

<h3 id="2008">2008</h3>
<p>Lauter, N., Moscou, M.J., Habinger, J., Moose, S. (2008) Quantitative genetic dissection of shoot architecture traits in maize: Towards a functional genomics approach. <b><i>The Plant Genome</i></b> <b>1</b>(2): 99-110 <a href="https://doi.org/10.3835/plantgenome2008.06.0385">doi:10.3835/plantgenome2008.06.0385</a></p>

<p>Potokina, E., Druka, A., Luo, Z., Moscou, M., Wise, R., Waugh, R., Kearsey, M. (2008) Tissue-dependent limited pleiotropy affects gene expression in barley. <b><i>Plant Journal</i></b> <b>56</b>(2): 287-296 <a href="https://doi.org/10.1111/j.1365-313X.2008.03601.x">doi:10.1111/j.1365-313X.2008.03601.x</a></p>

<h3 id="2007">2007</h3>
<p>Wise, R.P., Moscou, M.J., Bogdanove, A.J., Whitham, S.A. (2007) Transcript profiling in host–pathogen interactions. <b><i>Annual Review of Phytopathology</i></b> <b>45</b>: 329-369 <a href="https://doi.org/10.1146/annurev.phyto.45.011107.143944">doi:10.1146/annurev.phyto.45.011107.143944</a></p>

<h2 id="preprints">Preprints</h2>
<p>Clare, S., Kitcher, W., Gardiner, M., Green, P., Hubbard, A., and Moscou, M.J. (2016) Defining the genetic architecture of stripe rust resistance in the barley accession HOR 1428. <b><i>bioRxiv</i></b> 093773 <a href="https://doi.org/10.1101/093773">doi:10.1101/093773</a></p>

<h2 id="conference-publications">Conference Publications</h2>
<p>Champouret N, Moscou MJ, Bouyioukos C, Steuernagel B, Hernández-Pinzón I, Green P, Kaufman J, Olivera PD, Pretorius Z, Millet E, Steffenson BJ, Ward ER and Wulff BBH (2011) A pipeline for cloning resistance genes effective against African stem rust races from the diploid wheat relative Aegilops sharonensis In: Proc Borlaug Global Rust Initiative, June 13–19. McIntosh, R. (ed). pp. 66–74.</p>

<p>Wise, R. P., Meng, Y., Moscou, M. J., Xu, W. 2009. Regulators of innate immunity in cereal-fungal interactions. In Biology of Plant Microbe Interactions, Volume 7. International Society for Molecular Plant-Microbe Interactions, St. Paul, MN.</p>

<p>Moscou, M.J., Caldo, R.A., Lauter, N., Wise, R.P. 2008. Construction of coexpression networks to explore barley-powdery mildew interactions. Paper 113 in: Biology of Plant Microbe Interactions, Volume 6. M. Lorito, S. L. Woo, and F. Scala, eds. International Society for Molecular Plant-Microbe Interactions, St. Paul, MN.</p>

<h2 id="invited-talks">Invited Talks</h2>
<p>Accessing natural variation in Brachypodium distachyon: Development of the ABR6 x Bd21 RIL population. Plant and Animal Genome Conference. San Diego, CA, USA. January 17, 2017</p>

<p>Genomics of the barley immune system. Plant and Animal Genome Conference. San Diego, CA, USA. January 14, 2017</p>

<p>Engineering durable disease resistance. Gatsby Plant Science Network Meeting. Oxford, United Kingdom. September 9, 2016.</p>

<p>Dual specificity at the Mla locus confers resistance to barley powdery mildew and wheat stripe rust. International Powdery Mildew Symposium. Portland, Oregon, USA. July 16, 2016.</p>

<p>The barley ‘nibblerome’: Defining the set of NB-LRR-type R genes from a diverse collection of barley. International Barley Genetics Symposium. Minneapolis, Minnesota, USA. June 28, 2016.</p>

<p>Map-based cloning of host and nonhost resistance genes in barley. USDA-ARS, Cereal Crops Research Unit, North Dakota State University, Fargo, North Dakota, USA, December 1, 2015.</p>

<p>Dissecting the genetic architecture of nonhost resistance to stripe rust. Plant Biotic Stresses &amp; Resistance Mechanisms II, Vienna, Austria. July 3, 2015.</p>

<p>Dissecting the basis of host species specificity and nonhost resistance to wheat stripe rust. Heinrich-Heine-Universität Düsseldorf, Düsseldorf, Germany. November 24, 2014.</p>

<p>Nonhost resistance as a potential source for controlling wheat rust disease. International Congress on Molecular Plant-Microbe Interactions, Rhodes, Greece. July 10, 2014.</p>

<p>Harnessing the loci that determine the inappropriate host status of barley to engineer durable resistance to wheat stripe rust. North American Barley Researcher Workshop, Minneapolis, Minnesota, USA. July 2, 2014.</p>

<p>Why does wheat stripe rust not infect barley? Plant and Animal Genome Conference, San Diego, California, USA. January 11, 2014.</p>

<p>Deciphering the genetic architecture of host species specificity and nonhost resistance to wheat stripe rust. Oregon State University, Corvallis, Oregon, USA. January 8, 2014.</p>

<p>The genetic architecture underlying host species specificity of stripe rust. Washington State University, Pullman, Washington, USA. January 6, 2014.</p>

<p>Harnessing the loci underlying host species specificity to engineer durable resistance to wheat stripe rust. Keystone Symposia: Plant Immunity: Pathways and Translation. Big Sky, Montana, USA. April 7th-April 12th, 2013.</p>

<p>Host species specificity to stripe rust in barley colocalizes with loci conditioning resistance to powdery mildew. University of Minnesota, St. Paul, Minnesota, USA. March 28th, 2013</p>

<p>spst1, a novel susceptibility gene in barley to wheat stripe rust. 13th International Cereal Rust and Powdery Mildew Conference. Beijing, China. August 28th-September 1st, 2012.</p>

<p>Microscopy-based quantitative analysis of nonhost resistance in Brachypodium distachyon to cereal rusts. Monogram Conference. Aberystwyth, United Kingdom. March 26th-28th, 2012</p>

<p>Epistatic control of powdery mildew-induced necrosis among progeny of the Steptoe x Morex doubled-haploid mapping population in barley.  John Innes Centre (JIC), Norwich, United Kingdom. March 17, 2009</p>

<p>Natural Genetic Variation in Barley Identifies Master Regulatory Loci of Defense-Related Genes. Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Gatersleben, Germany. March 19, 2008</p>

<p>Gene-For-Gene-Mediated Transcriptome Reprogramming In Barley-Powdery Mildew Interactions. Plant and Animal Genome Conference, San Diego, California, USA. January 12, 2008.</p>

<p>Bioinformatics and Computational Biology Laboratory: An interaction model for cross-departmental bioinformatics knowledge integration (http://lab.bcb.iastate.edu). New Mexico State University, Las Cruces, New Mexico, USA. November 14, 2007.</p>

<p>PLEXdb: Plant and pathogen expression database and tools for comparative and functional genomics analysis. COST Action 858, INRA, Bordeaux, France. November 20, 2006</p>

<p>Markermaker: an iterative algorithm for optimization of fit between expression level polymorphisms and genetic marker datasets. Sixth Annual Joint Bioinformatics Symposium, Iowa State University, Ames, Iowa, USA. July 14, 2006.</p>

<h2 id="peer-review">Peer Review</h2>
<p>Manuscripts reviewed for BMC Genomics, BMC Plant Biology, Frontiers in Plant Science, Genome Biology, Journal of Experimental Botany, Molecular Genetics and Genomics, Molecular Plant Pathology, Nature Communications, Nature Genetics, Physiological and Molecular Plant Pathology, Phytopathology, Plant Cell Reports, Plant Health Progress, Plant Management Network, PLoS Genetics, PLoS ONE, Rice, Science, and Theoretical and Applied Genetics.</p>

<p><img src="https://raw.githubusercontent.com/matthewmoscou/matthewmoscou.github.io/master/_images/publons.png" alt="alt text" title="Publons" height="24px" width="24px" /> <a href="https://publons.com/a/602996/">https://publons.com/a/602996/</a></p>

<h2 id="membership-in-professional-societies">Membership in Professional Societies</h2>
<p>American Association for the Advancement of Science<br />
American Society of Plant Biologists<br />
British Society of Plant Pathology<br />
Genetics Society of America<br />
International Society of Molecular Plant-Microbe Interactions</p>

<p>Last updated: 2017 March 13</p>

  </div>

</article>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Moscou Group at The Sainsbury Laboratory</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>
            
              Moscou Group at The Sainsbury Laboratory
            
            </li>
            
            <li><a href="mailto:matthew.moscou@tsl.ac.uk">matthew.moscou@tsl.ac.uk</a></li>
            
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/matthewmoscou"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">matthewmoscou</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/MatthewMoscou"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">MatthewMoscou</span></a>

          </li>
          
        </ul>
      </div>

      <div class="footer-col footer-col-3">
        <p>The Moscou group focuses on understanding immunity in the grasses. Current themes in our research include dissecting the genetic architecture of nonhost resistance, tradeoffs between biotrophs and necrotrophs, seedling versus adult plant resistance, and suppression of resistance in polyploid genomes. The group uses genetics and bioinformatics to explore biology and solve long-standing puzzles.
</p>
      </div>
    </div>

  </div>

</footer>


  </body>

</html>
