---
layout: default
---

<!-- Main -->
<section id="banner">
	<header class="inner">
		<h2>{{ page.title }}</h2>
	</header>
</section>

<a name="nonhost"></a>
<section id="main" class="wrapper style1">
	<div class="container">
		<div class="row">
			<div class="4u">
				<section class="special">
					<a class="image fit"><img src="../images/morexpst.jpg" alt="" /></a>
				</section>
			</div>
			<div class="8u">
				<section class="specialleft">
					<h2>Genetic architecture of nonhost resistance in grasses to fungal pathogens</h2>
					<p><b>Collaborations:</b> Michael Ayliffe at <a href=https://www.csiro.au/en/Locations/ACT/Black-Mountain>CSIRO, Black Mountain</a>, Wolfgang Spielmeyer at <a href=https://www.csiro.au/en/Locations/ACT/Black-Mountain>CSIRO, Black Mountain</a>, Zakkie Pretorius <a href=http://www.ufs.ac.za/natagri/departments-and-divisions/plant-sciences-home>University of the Free State</a>, and Melania Figueroa at <a href=https://plpa.cfans.umn.edu/people/faculty/melania-figueroa>University of Minnesota</a></p>
					<p>Nonhost resistance has long been considered a source for engineering durable disease resistance. While we have an extensive mechanistic understanding of the genes underlying host resistance, little is known about the genes conditioning resistance in the transition from host to nonhost. We set out to understand the genetic architecture underlying resistance to the biotrophic fungal pathogen <i>Puccinia striiformis</i> f. sp. <i>tritici</i> (wheat stripe rust) in intermediate host and nonhost grass species. In the diploid grass species <i>Hordeum vulgare</i> (barley) and <i>Brachypodium distachyon</i> (purple false brome) we have identified several loci that contribute to limiting the host range of <i>P. striiformis</i>. Mapping of nonhost resistance in both species has found that resistance is conditioned by a few major effect loci. We have fine-mapped six nonhost resistance loci, including <i>Rps6</i>, <i>Rps7</i>, and <i>Rps8</i> in barley, and <i>Yrr1</i>, <i>Yrr2</i>, and <i>Yrr3</i> in <i>B. distachyon</i>. Ongoing work in the group includes map-based cloning of these loci and their association with known and novel components of immunity.</p>
				</section>
			</div>
		</div>
	</div>
</section>

<a name="multiplerecognition"></a>
<section id="main" class="wrapper style2">
	<div class="container">
		<div class="row">
			<div class="8u">
				<section class="specialleft">
					<h2>Multiple pathogen recognition of <i>R</i> genes</h2>
					<p><b>Collaborations:</b> Tom Wolpert at <a href=http://bpp.oregonstate.edu/wolpert>Oregon State University</a>, Ryohei Terauchi at <a href=http://www.kais.kyoto-u.ac.jp/english/overview/members/>Kyoto University</a>, and Ulrich Schaffrath at <a href=http://www.bio3.rwth-aachen.de/>RWTH Aachen University</a></p>
					<p>While the genetic coupling of disease resistance is common among several plant-pathogen systems, relatively few <i>R</i> genes loci have been shown to confer resistance to multiple pathogens is conditioned within the same haplotype. Known examples include the <i>RPS4</i>/<i>RRS1</i> <i>R</i> gene pair in <i>Arabidopsis thaliana</i>, <i>Mi-1</i> in tomato, and the adult plant resistance gene <i>Lr34</i> in wheat. We are currently expanding our understanding of the genetic coupling of <i>R</i> genes and their potential tradeoffs between biotrophs and necrotrophs (see next Research topic). Current work in the group involves the molecular dissection of the <i>Mla7</i>/<i>Mla8</i>/<i>Rps7</i>, <i>Mla3</i>/<i>HvLOV</i>/<i>Rmo1</i>, and <i>Pc2</i>/<i>Vb</i> complexes.
					</p>
				</section>
			</div>

			<div class="4u">
				<section class="special">
					<a class="image fit"><img src="../images/inseperable_R_genes.png" alt="" /></a>
				</section>
			</div>
		</div>
	</div>
</section>

<a name="tradeoffs"></a>
<section id="main" class="wrapper style1">
	<div class="container">
		<div class="row">
			<div class="4u">
				<section class="special">
					<a class="image fit"><img src="../images/biotoneclowres.jpg" alt="" /></a>
				</section>
			</div>
			<div class="8u">
				<section class="specialleft">
					<h2>Molecular tradeoffs in immunity to biotrophs and necrotrophs</h2>
					<p><b>Collaborations:</b> Tom Wolpert at <a href=http://bpp.oregonstate.edu/wolpert>Oregon State University</a></p>
					<p>Breeding for disease resistance often involves the introgression of resistance (<i>R</i>) genes into susceptible commercial crop cultivars. During the 1940s, oat crown rust (<i>Puccinia coronata</i>) resistance derived from the oat (<i>Avena sativa</i>) cultivar Victoria was introduced into elite germplasm. Subsequently, Victoria and its derivatives were grown throughout the US Midwest. Unbeknownst to plant breeders, resistance conferred by the <i>R</i> gene <i>Pc2</i> and its introduction facilitated a host jump of the necrotrophic pathogen <i>Cochliobolus victoriae</i> from timothy-grass (<i>Phleum pratense</i>) onto oat. This jump caused a massive epidemic, creating a situation where a novel disease spread across oat fields and caused farms to switch to completely different cultivars in subsequent years. Virulence of <i>C. victoriae</i> on oat requires two genetic factors: (1) the susceptibility gene <i>Victoria blight</i> (<i>Vb</i>), which is in coupling with <i>Pc2</i> and (2) the host-selective toxin victorin, which exists as several complex chlorinated cyclic pentapeptides. Oat is a difficult genetic system, but the discovery that victorin elicits sensitivity in several plant species including barley, <i>Brachypodium distachyon</i>, rice, <i>Phaseolus vulgaris</i>, and critically, <i>Arabidopsis thaliana</i>, meant that these systems could be used to understand how victorin induces susceptibility to necrotrophs. In <i>Arabidopsis thaliana</i>, it was shown that <i>LOV1</i> (<i>LOCUS ORCHESTRATING VICTORIN EFFECTS1</i>) conditioning victorin sensitivity encodes a coiled-coil, nucletotide binding, leucine-rich repeat (CC-NB-LRR) protein. NB-LRR genes are typically involved in disease resistance and this observation highlights a mechanism of exploitating the plant immune response to aid necrotrophic pathogen infection. Ongoing work in the group includes the identification of the genes controlling victorin sensitivity in barley (<i>HvLov1</i>) and <i>B. distachyon</i> (<i>BdLov1</i>). In addition, we are current taking a RenSeq/PacBio-based approach to mine the NB-LRRome of oat and use natural and induced variation to identify <i>Pc2</i>/<i>Vb</i>.</p>
				</section>
			</div>
		</div>
	</div>
</section>

<a name="suppression"></a>
<section id="main" class="wrapper style2">
	<div class="container">
		<div class="row">
			<div class="8u">
				<section class="specialleft">
					<h2>Suppression of resistance in polyploid genomes</h2>
					<p><b>Collaborations:</b> Colin Hiebert at <a href=http://www.agr.gc.ca/eng/science-and-innovation/research-centres/manitoba/morden-research-and-development-centre/scientific-staff-and-expertise/hiebert-colin-phd/?id=1272636174163>Agriculture and Agri-Food Canada</a> and Wolfgang Spielmeyer at <a href=https://www.csiro.au/en/Locations/ACT/Black-Mountain>CSIRO, Black Mountain</a></p>
					<p>Polyploidy is a widespread feature among plants and is associated with the improvement of a range of traits including biotic and abiotic stress, development, and life history traits. In allotetraploids, the presence of multiple divergent genomes creates a genetic landscape that can involve novel interactions between genomes. In 1980, Kerber and Green identified the presence of a suppressor of resistance in wheat to wheat stem rust (<i>Puccinia graminis</i> f. sp. <i>tritici</i>). Kerber and Green (1980) mapped the suppression to chromosome 7D, and Kerber (1991) established that a single locus conferred suppression on chromosome 7D. Current work in the group involves the identification of the <i>R</i> gene suppressor and the <i>R</i> genes that are activated.</p>

				</section>
			</div>
			<div class="4u">
				<section class="special">
					<a class="image fit"><img src="../images/suppression.jpg" alt="" /></a>
				</section>
			</div>
		</div>
	</div>
</section>

<a name="genomics"></a>
<section id="main" class="wrapper style1">
	<div class="container">
		<div class="row">
			<div class="4u">
				<section class="special">
					<a class="image fit"><img src="../images/phylogenetictreemedres.png" alt="" /></a>
				</section>
			</div>
			<div class="8u">
				<section class="specialleft">
					<h2>Genomics of immunity in cereals</h2>
					<p>Positional cloning of resistance (<i>R</i>) loci to biotrophic pathogens in angiosperms has revealed that the majority of genes encode proteins with two critical domains: a nucleotide binding (NB) domain and tandem arrays of leucine rich repeats (LRRs). Localized to the cytoplasm and nucleus, NB-LRRs recognize the presence of pathogens through direct recognition of pathogen-derived elicitors or through indirect recognition of host proteins that are modified by the pathogen. NB-LRR encoding loci are highly complex, exhibiting substantial copy number variation, structural rearrangements, and novel gene content between allelic variants. Collectively, these characteristics render this gene family a major challenge with modern assembly and annotation from next-generation sequencing. Using the Morex whole genome shotgun reference assembly as a starting point, we identified 328 NB-LRRs are present in the annotated Morex genome. The NB-LRRs are unequally distributed across the genome, with particular enrichment on chromosome arms: 1HS, 2HS, 6HL, and 7HS. We discovered that the annotated NB-LRRs were incomplete and set out to improve the annotation of this gene family. Using an approach that integrates transcriptome-based discovery pipeline and RenSeq/PacBio (an exome capture that specifically targets the NB-LRRs), we have identified over 600 NB-LRR encoding genes. We are now leveraging the leaf transcriptomes derived from 39 elite, landrace, and wild barley accessions to assess allelic diversity including phylogenetic relationship and degree of presence/absence variation relative to the reference genome.</p>
				</section>
			</div>
		</div>
	</div>
</section>

<a name="rgenes"></a>
<section id="main" class="wrapper style2">
	<div class="container">
		<div class="row">
			<div class="8u">
				<section class="specialleft">
					<h2>Resistance gene discovery in barley</h2>
					<p><b>Collaborations:</b> Brian Steffenson at <a href=https://plpa.cfans.umn.edu/people/faculty/brian-steffenson>University of Minnesota</a></p>
					<p>Barley stripe rust (=barley yellow rust), caused by <i>Puccinia striiformis</i> f. sp. <i>hordei</i>, is a destructive fungal pathogen that significantly affects worldwide barley cultivation. A major constraint in breeding resistant cultivars is the lack of mapping information of resistance (<i>R</i>) genes and their introgression into adapted germplasm. A considerable number of <i>R</i> genes have been described in barley to <i>P. striiformis</i> f. sp. <i>hordei</i>, but only a few loci have been mapped. Current work in the group involves the mapping of over 10 different <i>Resistance to P. striiformis</i> (<i>Rps</i>) and the generation of a near-isogenic panel using the recurrent cultivar Manchuria.</p>
				</section>
			</div>
			<div class="4u">
				<section class="special">
					<a class="image fit"><img src="../images/rdiscovery.png" alt="" /></a>
				</section>
			</div>
		</div>
	</div>
</section>

<a name="barriers"></a>
<section id="main" class="wrapper style1">
	<div class="container">
		<div class="row">
			<div class="4u">
				<section class="special">
					<a class="image fit"><img src="../images/segregation_distortion.png" alt="" /></a>
				</section>
			</div>
			<div class="8u">
				<section class="specialleft">
					<h2>Pre-zygotic and post-zygotic hybridization barriers</h2>
					<p><b>Collaborations:</b> Sebastian Schornack at <a href=http://www.slcu.cam.ac.uk/research/schornack-group>Sainsbury Laboratory, Cambridge University</a></p>
					<p>The transmission of alleles from one generation to the next involves several barriers that can restrict gene flow. Biological processes that can control prezygotic selection include asymmetric transmission of chromosomes in gamete development, gametophytic lethality, pollen viability, adhension, germination, and rejection such as allele-specific pollen tube growth and acceptance, fertilization of the ovule, and irregular endosperm development. After fertilization, genomic conflict is the primary form of postzygotic selection, leading to abortive zygotes or the observation of hybrid necrosis in F1 and F2 progeny with phenotypes that can include dwarfism, chlorosis, necrosis, or lethality. In developing genetic resources in barley for a range of projects, we identified a novel prezygotic locus, <i>Gametophyte factor2</i> (<i>Gaf2</i>), which controls the transmission of alleles in pollen, and the hybrid lethality (postzygotic) gene <i>Lethal progeny1</i> (<i>Lep1</i>). Current work in the group involves the fine-mapping of these loci and identification of candidate genes.</p>
				</section>
			</div>
		</div>
	</div>
</section>

<a name="resources"></a>
<section id="main" class="wrapper style2">
	<div class="container">
		<div class="row">
			<div class="8u">
				<section class="specialleft">
					<h2>Genetic resource development</h2>
					<p><b>Collaborations:</b> John Doonan at <a href=https://www.aber.ac.uk/en/ibers/staff-profiles/listing/profile/jhd2>Aberystwyth University</a> and Patrick Hayes at <a href=http://barleyworld.org/>Oregon State University</a></p>
					<p>Recombinant inbred line (RIL) and doubled-haploid (DH) mapping populations remain a critical resource for gene discovery. In both barley and <i>Brachypodium distachyon</i>, we are involved in the generation of several mapping populations for the community. In barley we are developing the Oregon Promise DH population (OPP), which is a cross between the 2-row malting (particularly for distilling) and transformable cultivar Golden Promise and the 2-row craft malting cultivar Full Pint. The OPP segregates for a range of morphological, disease resistance, and malting traits. In <i>B. distachyon</i> we are developing the ABR6 x Bd21 RIL population, which represents an extreme cross between the Spanish accession ABR6 and the Iraqi accession Bd21 (also the gold-standard sequenced reference accession). These populations are open to the community, please contact us for access.</p>
				</section>
			</div>
			<div class="4u">
				<section class="special">
					<a class="image fit"><img src="../images/brachypodium.jpg" alt="" /></a>
				</section>
			</div>
		</div>
	</div>
</section>

